{{Oxford/Loader}}
{{Oxford}}
<html lang="en">

<!-- THIS IS WHERE THE HTML BEGINS -->

<body>
  
    <section id="banner">
        <div class="container" style="width:100%; padding-right:0; padding-left:0;">
          <div class="row">
            <div class="col-md-12">
          <div class="inner" style="background-image: url('https://2019.igem.org/wiki/images/a/a4/T--Oxford--DS_Wet_Lab_Photo.jpeg'); background-size: cover; background-position:center">
            <div class=image_faded_overlay" style="background-color: lightskyblue; opacity:0.2">
            <div class="translucent-back" style="height: 400px;">
            </div>
          </div>
          <div class="banner_overlay_text">
            Experiments
          </div>
          </div>
  
        </div>
      </div>
    </div>
        </section>

        <section id="n1" style="padding-bottom: 15px;">
            <div class="container">
              <div class="row">
                <div class=".col-md-8 .offset-md-2">

                  <h2><i>E. coli</i></h2>

                  <h3>Strain of Choice</h3>

                  <h4><i>Escherichia coli</i> strains are commonly used host organisms in labs because they have several advantages: cheap, replicate rapidly, have well characterized and optimized systems, etc. which make them very convenient for lab work. </h4>
                  <h4>In our project, we opted for using <i>E. coli</i> as well when we had to amplify plasmid DNA, express proteins for purification or just for testing constructs before they were transformed into <i>L. reuteri</i>. </h4>
                  <h4>We mainly used two strains for our <i>E. coli</i> work: NEBTurbo competent cells for cloning, and BL21(DE3)-RIPL competent cells for protein expression.</h4>

                  </div>
                  </div>
                  </div>
        </section>

        <section id="n2" style="background-color: #f2f2f2ff; padding-bottom:15px;">
            <div class="container">
              <div class="row">
                <div class=".col-md-8 .offset-md-2">

                  <h3>Gibson Assembly of Constructs</h3>

                  <h4>For our project, we designed several constructs which were used for various experiments. To assemble these constructs, we used the protocol developed by Gibson et al. (2009)<sup>1</sup>. </h4>

                  <h4>The Gibson assembly (GA) offers a quick and easy method to construct DNA fragments and recombinant plasmids. This method uses a DNA exonuclease that creates single stranded DNA overhangs which can anneal with other complementary overhangs, DNA polymerase that fills in any gaps, and DNA ligase that connects the DNA fragments together, all in one reaction mixture<sup>1</sup>.</h4>

                  <img src="" style="width:75%; display:block; margin-left: auto; margin-right: auto;" alt="Figure 1: Overview of Gibson Assembly">

                  <p style="font-size:14px;">Figure 1. Overview of Gibson Assembly. (Gibson Assembly by NivinN is licensed under CC BY 4.0)</p>

                  <h4>This method was an alternative to using restriction enzymes. We used the Gibson Assembly to construct all of our recombinant plasmids. Having our main inserts synthetized by IDT, we only needed to clone the fragment into the corresponding vector. This was achieved by creating overhangs using appropriate primers in polymerase chain reactions (PCR) in the inserts and the vector backbones. The resulting fragments were separated by agarose gel electrophoresis to purify only those bands that corresponded to our fragment of interest. The purified DNA was then used for Gibson Assembly, which linked the fragments together creating the final recombinant plasmid constructs.</h4>

                  <h4>Moreover, this method enabled us to easily troubleshoot our assemblies simply by ordering new primers. We have encountered such issues, notably when we needed a construct with only a promoter and a reporter gene (e.g. erm-mClover3,ldh-mClover3) or when one of the constructs was missing terminators due to the complex secondary structures that would have been difficult for IDT to synthesise. Instead, we were able to attach such terminators, using PCR and Gibson assembly, from another construct.</h4>

                  <h4>The products of Gibson Assembly reactions were transformed into <i>E. coli</i> (NEBTurbo cloning strain) competent cells, selected on agar plates, and correct assembly of product was validated by sequencing.</h4>


                  </div>
                  </div>
                  </div>
        </section>

        <section id="n3" style="padding-bottom: 15px;">
            <div class="container">
              <div class="row">
                <div class=".col-md-8 .offset-md-2">

                  <h3>Expression +  Purification of CD27L & Truncated Endolysins for Killing Assay</h3>

                  <h4>One of the main components of our experimental work is the killing of <i>C. difficile</i> by using CD27L, an endolysin from ɸCD27 bacteriophage. CD27L is a protein with two domains: an N-terminal N-acetylmuramoyl-L-alanine amidase homologue and a C-terminal domain. It has been argued that the N-terminal domain has the same specificity as the full length CD27L and improved catalytic activity<sup>2</sup>.</h4>

                  <h4>We were interested in testing this endolysin in different conditions, to see how it behaves in different environments. Thus, to carry out those experiments, we expressed both the CD27L and CD27L<sub>1-179</sub> endolysins, allowing us to compare them and test how they behave.</h4>

                  <h4>For our purification strategy, we expressed the endolysins in <i>E. coli</i> and purified it via  immobilized metal ion affinity chromatography (IMAC) to obtain a partially purified protein solution. Then, after confirming by SDS-PAGE that we had the correct product, we dialysed our protein into phosphate buffered saline (PBS). With such a purification strategy, we aimed to obtain a pure enough sample of active endolysin for killing  assays.</h4>

                  <h4>To express our endolysin, we fused our endolysin with a N-terminal 6x-His tag and a SpyTag<sup>3</sup>, which enables the utilisation of two different affinity chromatography (AC) methods, as well as tandem affinity purification (TAP). </h4>

                  <h4>The constructed part was cloned into an IPTG-inducible, T7 expression system-based pET28a vector. For our host organism, we used the BL21(DE3)-RIPL expression strain, which is optimized for the T7 expression system and rare codons. We grew our cells in LB at 37&#8451; and then induced the cultures in mid-log phase with IPTG at 180RPM 21&#8451;. Following induction overnight, the cells were harvested using centrifugation and lysed by sonication. Purification was then carried out as mentioned above.</h4>

                  <h4>Our choice of buffer was based on previous kill assays in literature, which used PBS as the main buffer for the reaction<sup>2</sup>. Specifically, in cases where our yield was low (i.e. CD27L<sub>1-179</sub> construct) we used spin concentration to achieve total protein concentration that is suitable for assays. </h4>

                  <h4>During purification, we encountered several issues with both the CD27L and CD27L<sub>1-179</sub> proteins. The latter was expressed at insufficiently high concentrations for assaying, so spin concentration was used to achieve high enough concentrations.</h4>

                  <h4>Additionally, our progress with CD27L was also hindered by the precipitation of our samples during dialysis. Specifically, our CD27L endolysin precipitated in substantial quantities (protein concentration decreasing from 5mg/ml to 0.3mg/ml) in three batches of our five.</h4>

                  <h4>Our initial hypothesis was that this occurred due to the lack of reducing conditions. This hypothesis was based on the structural model of CD27L, where it is shown there are several surface cysteine residues. However, adding beta-mercaptoethanol to all steps of purification, including dialysis, did not resolve this issue. Thus, to fully address this question, further investigation is required.</h4>

                  <h4>Tandem Affinity Purification (TAP) was also planned to obtain higher purity samples. However, we decided that our one step affinity chromatography yielded sufficient purity for our particular CD27L applications.</h4>


                  </div>
                  </div>
                  </div>
        </section>

        <section id="n4" style="background-color: #f2f2f2ff; padding-bottom:15px;">
            <div class="container">
              <div class="row">
                <div class=".col-md-8 .offset-md-2">

                  <h3>CD27L Endolysin Assay</h3>

                  <h4>To test the CD27L endolysin, we had to create an assay demonstrating whether it is able to function as it was shown in literature<sup>2,4</sup>. From such preliminary research, we discovered that CD27L, besides being active against <i>Clostiridoides difficile</i>, is also able to cleave amide bonds in the cell wall of other bacteria with the same peptidoglycan type, A1γ. Thus, we used <i>Bacillus subtilis</i>, a Category 1 Gram-positive bacteria<sup>2</sup> as our surrogate target due to safety concerns with <i>C. difficile</i></h4>
                  <h4>In order to assay whether <i>B. subtilis</i> is lysed by our endolysin, we measured absorbance at 600nm (OD600) throughout an 8 hour time period at every 15 minutes. This way, we were able to monitor how the <i>B. subtilis</i> culture is affected by CD27L. </h4>
                  <h4>Notably, our kill assay differs from the one described by Mayer et al. (2008)<sup>4</sup>. Their protocol involved washing and concentrating the cells to OD600 = 2.0 in Phosphate Buffered Saline (PBS), followed by mixing with CD27L and measuring the rate of decrease in OD600. </h4>
                  <h4>However, our protocol involved growing up a <i>B. subtilis</i> culture until they reached mid-log phase (OD600 ≈ 0.6), then diluting them in Lysogenic Broth (LB), adding our solution (CD27L or Lysozyme or PBS) and then measuring the growth over time by monitoring the change in OD600. The measurements were made automatically by a plate reader, BMG FLUOstar Omega and the cells were grown in a 96 well plate from Greiner (CellStar® 96 well plate F-type). </h4>

                <h4>The use of a plate reader allowed us to measure the absorbance at OD600 over a long time period, which greatly reduces the possible human error. Additionally, the instrument allowed us to conduct experiments on a larger scale than otherwise possible. Notably, it is capable  of incubating the cells at 37 °C and shaking at 500 rpm, which helped to carry out the experiments in a reproducible manner. Furthermore, to increase confidence in our results, our experiments were always carried out in triplicates so that we could perform statistical analysis from our results.
                  </h4>



                  </div>
                  </div>
                  </div>
                </section>


      <section id="l1" style="padding-bottom: 15px;">
        <div class="container">
          <div class="row">
            <div class=".col-md-8 .offset-md-2">
  
              <h2><i>Lactobacillus reuteri</i></h2>

              <h3>Introduction</h3>
              
              <h4><i>Lactobacillus reuteri</i> is a well characterized probiotic bacterial species and the chassis of choice for our ProQuorum system. Some of the criteria we considered essential for the strain included:</h4>

              <ol style="font-size:18px;">
                <li>the ability to survive and proliferate in the human gut</li>
                <li>the potential for efficient endolysin production and secretion</li>
                <li>the capability to express the two-component signalling system functionally</li>
              </ol>

              <h4>Regarding the first criterion, many of the <i>Lactobacillus</i> species have been shown to successfully colonise the gut and even promote its normal physiology, while also inhibiting the proliferation of specific pathogenic species<sup>1</sup>. </h4>

              <h4>Regarding the second criterion, the genus has the added advantage of belonging to the group of Gram-positive bacteria, which we thought to be relevant for the secretion of the CD27L endolysin. This is given the lack of the potential barrier of the outer membrane of Gram-negative bacteria. At the same time, the lack of an outer membrane would facilitate the diffusion of the autoinducer peptide (AIP) of <i>C. difficile</i> (indeed, itself a Gram positive species) to the AgrC receptor. </h4>

              <h4>Regarding the third criterion, <i>L. reuteri</i> has been shown to express the AgrAC system and be induced by AIP previously<sup>2</sup>. As part of our project, we aimed to show that the CD27L endolysin can be expressed and released into the extracellular milieu by <i>L. reuteri</i> and that AgrAC can be expressed and carry out its normal function.</h4>
            <br>

            <div class="panel-group">
                <div class="panel panel-default">
                <div class="panel-heading">
                  <h4 class="panel-title">
                    <a data-toggle="collapse" href="#Electroporation">Transformation via Electroporation</a>
                  </h4>
                </div>
                <div id="Electroporation" class="panel-collapse collapse">
                  <centre><div class="panel-body">
                      <h4>While the transformation process was initially envisioned as a minor step that would enable us to perform more relevant assays on our system such as killing assays, it became a significant part of our project after many weeks with no positive results. </h4>

                      <h4>Initially, we attempted to work with the <i>L. reuteri</i> strain DSM20016,; however, after unsuccessfully attempting every electroporation protocol for the species we could find in the literature, we ultimately discovered that following uptake of our <i>E. coli</i>-derived miniprep DNA, our transformed DNA was digested by <i>L. reuteri</i> nucleases. This was due to the particular methylation patterns on our <i>E. coli</i>-derived DNA. In our endeavour to resolve this issue, we turned to the Quadram Institute in Norwich, who have kindly helped us by providing a <i>Lactobacillus reuteri</i> 100-23c strain, for which transformation worked seamlessly.  </h4>
      
          </div></centre>
      </div>
      </div>
      </div>

      <div class="panel-group">
          <div class="panel panel-default">
          <div class="panel-heading">
            <h4 class="panel-title">
              <a data-toggle="collapse" href="#L_reuteri_CD27l">CD27L Endolysin</a>
            </h4>
          </div>
          <div id="L_reuteri_CD27l" class="panel-collapse collapse">
            <centre><div class="panel-body">
                <h4>For our experimentation with the CD27L endolysin, our intended course of action was to constitutively express the CD27L endolysin (due to the scarcity of options available for inducible expression vectors in <i>Lactobacillus spp.</i>). We would then assay its relative abundance inside and outside the cell. This would be determined via in gel SpyCatcher detection and/or by Western blot using anti-hexahistidine antibodies. </h4>
                <h4>In addition, endolysin activity assays were planned to prove that cellular processing in the new host would not alter the enzyme’s regular functionality, as well as to assess whether therapeutically effective activity would be reached in the extracellular medium. </h4>
                <h4>While attempts to express the protein in our chassis have failed, we have shown that constitutive protein expression brings about significant cellular stress; this has been determined by a combination of growth assays, fluorometry and fluorescence microscopy. Though such results have not been desirable, they open new avenues for approaching for expression strategies to limit cellular stress and strongly support the use of an AIP-inducible system.</h4>

    </div></centre>
</div>
</div>
</div>

<div class="panel-group">
    <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" href="#L_reuteri_AgrAC">AgrAC Two-Component Regulatory System</a>
      </h4>
    </div>
    <div id="L_reuteri_AgrAC" class="panel-collapse collapse">
      <centre><div class="panel-body">
          <h4>AgrC and AgrA are the proteins of the <i>C. difficile</i> quorum sensing signalling system responsible for AIP detection and transcriptional regulation, respectively. Their expression should be constant in the ProQuorum strain, such that we opted for their regulation by slp, a weak, yet constitutively active promoter<sup>3</sup>.</h4>
          <h4>As part of our plans, we intended to produce a reporter system containing the AgrAC couple, as well as a fluorescent reporter under the control of the Agr promoter. This reporter system would be induced by AIP., and thus validate the AgrAC regulatory system in <i>L. reuteri</i>.</h4>
          <h4>In addition, we intended to detect intermediary states in the signal transduction pathway, such as the phosphorylated form of AgrA, with the aid of phospho-aspartyl specific antibodies. However, similarly to the case of CD27L endolysin, the detection of expression failed.</h4>

</div></centre>
</div>
</div>
</div>

<div class="panel-group">
    <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" href="#L_reuteri_AIP">Autoinducer Peptide (AIP) Isolation & Detection</a>
      </h4>
    </div>
    <div id="L_reuteri_AIP" class="panel-collapse collapse">
      <centre><div class="panel-body">
          <h4>Another workstream of our project involved the purification of AIP for the AgrAC induction assays. For this, the 2019 Nottingham iGEM team kindly provided us with sterilised <i>C. difficile</i> culture supernatant, from which we attempted to extract AIP by acetone precipitation<sup>4</sup> and then detect it by mass spectrometry (MS). We attempted to enhance the MS results through the reaction of the AIP thiolactone with hydroxylamine. </h4>
</div></centre>
</div>
</div>
</div>

<div class="panel-group">
    <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" href="#L_reuteri_Prom">Promoter Comparison</a>
      </h4>
    </div>
    <div id="L_reuteri_Prom" class="panel-collapse collapse">
      <centre><div class="panel-body">
          <h4>To tailor our system to the optimal parameters derived from our mathematical model of the ProQuorum <i>L. reuteri</i> strain, we designed an assay for the measurement of the erm and ldh promoters’ strength. Such information would be essential for fine-tuning the expression of all our parts, allowing us to strike a balance between the need to obtain high CD27L yields and the need to minimise cellular stress on the chassis. </h4>
</div></centre>
</div>
</div>
</div>

    </div> 
      
</div>
</div>
</section>

<section id="l2" style="background-color: #f2f2f2ff; padding-bottom:15px;">
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">
          <h3>The Constructs</h3>

          <h4>For all our <i>L. reuteri</i> work we used the pTRKH3 vector, a plasmid specifically designed for efficient protein expression in <i>Lactobacillus spp.</i><sup>3</sup>, which we have purchased from Addgene. All constructs have been assembled by performing Gibson assembly on PCR amplified template synthesized by IDT.</h4>
          <br>

          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_SLPMOD">slpMOD_CD27L (BBa_K3183300)</a>
                </h4>
              </div>
              <div id="L_reuteri_SLPMOD" class="panel-collapse collapse">
                <centre><div class="panel-body">
                    <img src="https://2019.igem.org/wiki/images/1/1a/T--Oxford--slpMOD_CD27L.png" style="width:100%">
          </div></centre>
          </div>
          </div>
          </div>

          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_AgrAC_Construct">AgrAC (BBa_K3183102)</a>
                </h4>
              </div>
              <div id="L_reuteri_AgrAC_Construct" class="panel-collapse collapse">
                <centre><div class="panel-body">
                    <img src="https://2019.igem.org/wiki/images/2/2e/T--Oxford--AgrAC.png" style="width:100%">
          </div></centre>
          </div>
          </div>
          </div>

          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_ermGFP_Construct">erm-GFP (BBa_K3183100)</a>
                </h4>
              </div>
              <div id="L_reuteri_ermGFP_Construct" class="panel-collapse collapse">
                <centre><div class="panel-body">
                    <img src="https://2019.igem.org/wiki/images/d/d6/T--Oxford--Erm_GFP.png" style="width: 100%;">
          </div></centre>
          </div>
          </div>
          </div>

          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_ldh_Construct">ldh-mClover (BBa_K3183101)</a>
                </h4>
              </div>
              <div id="L_reuteri_ldh_Construct" class="panel-collapse collapse">
                <centre><div class="panel-body">
                    <img src="https://2019.igem.org/wiki/images/7/71/T--Oxford--ldh_mClover.png" style="width: 100%;">
          </div></centre>
          </div>
          </div>
          </div>

          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_Terminology">Terminology</a>
                </h4>
              </div>
              <div id="L_reuteri_Terminology" class="panel-collapse collapse">
                <centre><div class="panel-body">

                    <div class="table-responsive">
                        <table class="table">
                          <thead>
                            <tr>
                              <th>Name</th>
                              <th>Description</th>
              
                            </tr>
                          </thead>
                          <tbody>
                            <tr>
                              <td>erm promoter</td>
                              <td>Constitutive <i>Lactobacillus</i>-specific promoter</td>
                            </tr>

                            <tr>
                                <td>ldh promoter</td>
                                <td>Constitutive <i>Lactobacillus</i>-specific promoter</td>
                              </tr>

                              <tr>
                                  <td>slp promoter</td>
                                  <td>Weak, constitutive <i>Lactobacillus</i>-specific promoter</td>
                                </tr>

                                <tr>
                                    <td>6His</td>
                                    <td>Hexahistidine tag, used for protein purification</td>
                                  </tr>

                                  <tr>
                                      <td>SpyTag</td>
                                      <td>Purification and affinity tag</td>
                                    </tr>

                                    <tr>
                                        <td>HA</td>
                                        <td>Haemagglutinin tag, used for immunological detection</td>
                                      </tr>

                                      <tr>
                                          <td>slpMOD<sup>5</sup></td>
                                          <td><i>L. reuteri</i>-optimised secretion tag</td>
                                        </tr>

                                        <tr>
                                            <td>CD27L</td>
                                            <td>Bacteriophage-derived <i>C. difficile</i>-specific endolysin</td>
                                          </tr>

                                          <tr>
                                            <td>mClover3</td>
                                            <td>Fluorescent reporter gene</td>
                                          </tr>

                                          <tr>
                                            <td>GFP</td>
                                            <td>Fluorescent reporter gene</td>
                                          </tr>
                                          
                                          <tr>
                                            <td>AgrA</td>
                                            <td>Signal transducer & transcriptional activator</td>
                                          </tr>

                                          <tr>
                                            <td>AgrC</td>
                                            <td>Membrane protein, the AIP receptor and AgrA-directed kinase</td>
                                          </tr>
              
              
                          </tbody>
                        </table>
                      </div>
               
          </div></centre>
          </div>
          </div>
          </div>
        


          </div>
        </div>
        </div>
        </section>

     

    

          

            <section id="n6">>
                <div class="container">
                  <div class="row">
                    <div class=".col-md-8 .offset-md-2">
        
                        <h2 style="text-align: center;">
                          References
                          </h2>
        
                        <div class="table-responsive">
                          <table class="table">
                            <thead>
                              <tr>
                                <th> #</th>
                                <th>Reference</th>
                              </tr>
                            </thead>
                            <tbody>

                              <tr>
                                <td>1</td>
                                <td>Gibson, Daniel G., et al. “Enzymatic Assembly of DNA Molecules up to Several Hundred Kilobases.” Nature Methods, vol. 6, no. 5, May 2009, pp. 343–45. www.nature.com, doi:10.1038/nmeth.1318.</td>
                              </tr>

                              <tr>
                                <td>2</td>
                                <td>Mayer, M. J., et al. “Structure-Based Modification of a Clostridium Difficile-Targeting Endolysin Affects Activity and Host Range.” Journal of Bacteriology, vol. 193, no. 19, Oct. 2011, pp. 5477–86. DOI.org (Crossref), doi:10.1128/JB.00439-11.</td>
                              </tr>

                              <tr>
                                <td>3</td>
                                <td>Reddington, Samuel C., and Mark Howarth. “Secrets of a Covalent Interaction for Biomaterials and Biotechnology: SpyTag and SpyCatcher.” Current Opinion in Chemical Biology, vol. 29, Dec. 2015, pp. 94–99. DOI.org (Crossref), doi:10.1016/j.cbpa.2015.10.002.</td>
                                
                              </tr>

                              <tr>
                                <td>4</td>
                                <td>Mayer, Melinda J., et al. “Molecular Characterization of a Clostridium Difficile Bacteriophage and Its Cloned Biologically Active Endolysin.” Journal of Bacteriology, vol. 190, no. 20, Oct. 2008, pp. 6734–40. jb.asm.org, doi:10.1128/JB.00686-08.</td>

                            
                              </tr>

                              <tr>
                                <td>5</td>
                                <td>Anuar, Irsyad N. A. Khairil, et al. “Spy&Go Purification of SpyTag-Proteins Using Pseudo-SpyCatcher to Access an Oligomerization Toolbox.” Nature Communications, vol. 10, no. 1, Apr. 2019, pp. 1–13. www.nature.com, doi:10.1038/s41467-019-09678-w.</td>

                              </tr>

                              <tr>
                                <td>6</td>
                                <td>Darkoh, Charles, et al. “Toxin Synthesis by Clostridium Difficile Is Regulated through Quorum Signaling.” MBio, vol. 6, no. 2, May 2015, pp. e02569-14. mbio.asm.org, doi:10.1128/mBio.02569-14.</td>
                              </tr>

                              <tr>
                                <td>7</td>
                                <td>Sonnhammer, E. L., et al. “A Hidden Markov Model for Predicting Transmembrane Helices in Protein Sequences.” Proceedings. International Conference on Intelligent Systems for Molecular Biology, vol. 6, 1998, pp. 175–82.</td>

                              </tr>

                              <tr>
                                <td>8</td>
                                <td>Thoendel, Matthew, and Alexander R. Horswill. “Identification of Staphylococcus Aureus AgrD Residues Required for Autoinducing Peptide Biosynthesis.” Journal of Biological Chemistry, vol. 284, no. 33, Aug. 2009, pp. 21828–38. www.jbc.org, doi:10.1074/jbc.M109.031757.</td>
                              </tr>

                              <tr>
                                <td>9</td>
                                <td>Bajar, Bryce T., et al. “Improving Brightness and Photostability of Green and Red Fluorescent Proteins for Live Cell Imaging and FRET Reporting.” Scientific Reports, vol. 6, no. 1, Feb. 2016, p. 20889. DOI.org (Crossref), doi:10.1038/srep20889.</td>
                              </tr>
                              <tr>
                                <td>1</td>
                                <td>Jacobsen, C.N. et al., Screening of Probiotic Activities of Forty-Seven Strains of Lactobacillus spp. by In Vitro Techniques and Evaluation of the Colonization Ability of Five Selected Strains in Humans, Applied and Environmental Microbiology, Nov. 1999, p. 4949–4956</td>
                              </tr>
    
                              <tr>
                                <td>2</td>
                                <td> Lubcowicz D et al. (2018), Reprogramming Probiotic Lactobacillus reuteri as a Biosensor for Staphylococcus aureus Derived AIP-I Detection, ACS Synth Biol
                                  </td>
                              </tr>
                              <tr>
                                <td>3</td>
                                <td>Lizier, Michela, et al. “Comparison of Expression Vectors in Lactobacillus Reuteri  Strains.” FEMS Microbiology Letters, vol. 308, no. 1, Aug. 2010, pp. 8–15., doi:10.1111/j.1574-6968.2010.01978.x.</td>
                              </tr>

                              <tr>
                                <td>4</td>
                                <td>Darkoh, Charles, et al. “Toxin Synthesis by Clostridium Difficile Is Regulated through Quorum Signaling.” MBio, Feb. 2015.</td>
                                  
                              </tr>
                              
                              <tr>
                                <td>5</td>
                                <td>Fernandez, A., et al. “Enhanced Secretion of Biologically Active Murine Interleukin-12 by Lactococcus Lactis.” Applied and Environmental Microbiology, vol. 75, no. 3, May 2008, pp. 869–871., doi:10.1128/aem.01728-08.</td>
                              </tr>

                            </tbody>
                          </table>
                        </div>
                      </div>
                    </div>
                  </div>
                </section>

<style>
    .column {
  float: left;
  width: 25%;
  padding: 5px;
}
</style>

</body>
</html>
{{Oxford/Footer}}
