{{Oxford/Loader}}
{{Oxford}}
<html lang="en">

<!-- THIS IS WHERE THE HTML BEGINS -->

<body>
    <section id="banner">
        <div class="container" style="width:100%; padding-right:0; padding-left:0;">
          <div class="row">
            <div class="col-md-12">
          <div class="inner" style="background-image: url('https://2019.igem.org/wiki/images/6/6b/T--Oxford--Sponsors_Bear.jpeg'); background-size: cover;">
            <div class=image_faded_overlay" style="background-color: lightskyblue; opacity:0.2">
            <div class="translucent-back" style="height: 400px;">
            </div>
          </div>
          <div class="banner_overlay_text">
            Results
          </div>
          </div>
  
        </div>
      </div>
    </div>
        </section>

      <section id="n1" style="padding-bottom: 15px;">
        <div class="container">
          <div class="row">
            <div class=".col-md-8 .offset-md-2">
  
                <h2> Functionality of &Phi;CD27L Endolysin </h2>

                <h4>The CD27L endolysin is a critical component of our system, as it acts as the agent of <i>C. difficile</i> lysis. Three parts were tested throughout this sub-section of project—they are summarised for clarity in Table 1.</h4>
                <br>
              <h4>Table 1: Different Constructs of CD27L Endolysin in pET28A Expression Vector</h4>
                <div class="table-responsive">
                              <table class="table">
                                <thead>
                                  <tr>
                                    <th>Construct Name</th>
                                    <th>Construct</th>
                                    <th>Part Number</th>
                    
                                  </tr>
                                </thead>
                                <tbody>
                                  <tr>
                                    <td>Dundee-CD27L</td>
                                    <td><i>Hcp</i>-HA_Tag- CD27L<sub>1-179</sub></td>
                                    <td>BBa_K895004</td>
                                  </tr>

                                  <tr>
                                    <td>CD27L</td>
                                    <td>6His-SpyTag-CD27L</td>
                                    <td>BBa_K3183200</td>
                                  </tr>
                                    
                                  <tr>
                                    <td>CD27L<sub>1-179</sub></td>
                                    <td>6His-SpyTag-CD27L<sub>1-179</sub></td>
                                    <td>BBa_K3183201</td>
                                  </tr>
                    
                                </tbody>
                              </table>
                            </div>

                            <img src="https://2019.igem.org/wiki/images/6/65/T--Oxford--Taleof3Endolysins.png" style="display: block; width:75%; margin-left:auto; margin-right:auto;" alt="Tale of 3 Endolysins">
                            <p style="font-size: 14px;">Figure 0. Construct Diagrams of CD27L Endolysin</p>

          <h4>We wanted to optimise the killing efficacy of the endolysin. Thus, we sought out the endolysin in an existing part made by the 2012 iGEM Team Dundee. This part (Dundee-CD27L<sub>1-179</sub>) was comprised of CD27L<sub>1-179</sub> endolysin linked to the hcp Type VI secretion tag via an haemagglutinin (HA) epitope tag <a href=”http://parts.igem.org/Part:BBa_K895004”>(BBa_K895004)</a>. </h4>
          <h4>After multiple attempts at expression, we were unable to express the protein, with further information on the <a href=”https://2019.igem.org/Improve>Part Improvement</a> page.  Fig. 1 provides a comparison between expression of the full-length endolysin, CD27L and the 2012 Dundee iGEM part BBa_K895004.</h4>
          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#CD27L_Fig1">Fig 1: SDS-PAGE Gel Comparing CD27L Endolysin Samples with Dundee-CD27L Samples</a>
                </h4>
              </div>
              <div id="CD27L_Fig1" class="panel-collapse collapse">
                <centre><div class="panel-body">
                    <img src="https://parts.igem.org/wiki/images/3/3c/BBa_K895004_Expression_Gel.png" style="width: 100%;" alt="Fig 1: SDS-PAGE Gel Comparing CD27L Endolysin Samples with Dundee-CD27L Samples">
          </div></centre>
          </div>
          </div>
          </div>

          <h4>Given this outcome, we decided to express only the truncated CD27L<sub>1-179</sub> endolysin, removing the <i>hcp</i> Type VI secretion tag which we assumed to have interfered with expression.  Consequently,, successful expression occurred for this new part, BBa_K31830012. </h4>

          <h4>In conjunction, we decided to see if adding back the C-terminal domain, thought to be important for specificity, would affect the endolysin’s ability to lyse cells. Expression of this part, CD27L (BBa_K3183009) and CD27L<sub>1-179</sub> are shown in Figure 2.1 and 2.2.</h4>

          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#CD27L_Fig2">Figure 2: SDS-PAGE  Protein Purification Gels of CD27L & CD27L<sub>1-179</sub> using Ni-NTA Affinity Chromatography</a>
                </h4>
              </div>
              <div id="CD27L_Fig2" class="panel-collapse collapse">
                <centre><div class="panel-body">
                      <div class="container">
                        <div class="row">
                          <div class="col-md-6">
                              <img src="https://parts.igem.org/wiki/images/5/5d/BBa_K3183009_Purification_Gel.png" style="width: 100%;" alt="Fig. 3.1: SDS-PAGE Protein Purification Gel of CD27L using Ni-NTA Affinity Chromatography">
                          </div>
                         
                          <div class="col-md-6">
                              <img src="https://parts.igem.org/wiki/images/8/8c/BBa_K31830012_Purification_Gel.png" style="width: 100%;" alt="Fig. 3.2: SDS-PAGE Protein Purification Gel of CD27L<sub>1-179</sub> using Ni-NTA Affinity Chromatography">


                          </div>
                        </div>
                      </div>

                    </div></centre>
                  </div>
                  </div>
                  </div>

                  <h4>To confirm the identity of our endolysin, we wanted to run it through electrospray mass spectrometry (ESI-MS). However, given that we didn’t have access to size-exclusion chromatography, we were only able to send our CD27L protein preparation, as the other sample was far too dirty for effective ESI-MS. The identity of our endolysin, as well as putative autocleavage is shown and discussed in Fig. 3.</h4>
                  <br>
                  <div class="panel-group">
                      <div class="panel panel-default">
                      <div class="panel-heading">
                        <h4 class="panel-title">
                          <a data-toggle="collapse" href="#CD27L_Fig3">Fig. 3: Mass Spectrometry of CD27L Endolysin Following Ni-NTA Purification</a>
                        </h4>
                      </div>
                      <div id="CD27L_Fig3" class="panel-collapse collapse">
                        <centre><div class="panel-body">
                            <img src="https://2019.igem.org/wiki/images/f/fd/T--Oxford--MassSpectrumResultsPage.png" style="width: 100%;" alt="Fig. 2: Mass Spectrometry of CD27L Endolysin Following Ni-NTA Purification">
                          <h4>This mass spectrometry shows our protein clearly at 32979.68amu. However, Additional peaks are present at 9586.14amu and 16490.55. We believe that this is cleavage between the N-terminal (NTD) and C-terminal (CTD) domains. We cannot determine exactly why the peak at the NTD is much lower than the CTD, but a large band at 9.5kDa is present in our SDS-PAGE gels (Fig. 2.1). While this may be due to protease contamination, it has been suggested to be due to autocatalytic cleavage<sup>1</sup>.
                          </h4>
                          </div></centre>
                  </div>
                  </div>
                  </div>

                  <h4>Due to safety concerns, our endolysin killing assays could only be carried out on on <i>Bacillus subitlis</i>, a Category 1 bacteria as opposed to <i>Clostridium difficile</i>, a Category 2 bacteria. <i>B. subtilis</i> has a similar composition of peptidogylcan cell wall to <i>C. difficile</i><sup>2</sup>, allowing us to quantify killing data. </h4>
                   <h4>Figure 4 shows that the CD27L full-length endolysin has greater killing efficacy than the truncated CD27L<sub>1-179</sub>. However, this may be due to a much purer preparation of CD27L compared to CD27L<sub>1-179</sub> as seen in Fig. 3.</h4>
                     <br>

                     <div class="panel-group">
                        <div class="panel panel-default">
                        <div class="panel-heading">
                          <h4 class="panel-title">
                            <a data-toggle="collapse" href="#CD27L_Fig4">Fig. 4: CD27L Versus CD27L<sub>1-179</sub> Endolysin Killing Activity</a>
                          </h4>
                        </div>
                        <div id="CD27L_Fig4" class="panel-collapse collapse">
                          <centre><div class="panel-body">
                            
                            <img src="https://2019.igem.org/wiki/images/f/f8/T--Oxford--TruncatedEndolysinResults.png" style="width:100%" alt="Fig. 4: CD27L Versus CD27L<sub>1-179</sub> Endolysin Killing Activity">
                            <h4>The truncated CD27L<sub>1-179</sub> shows decreased growth relative to the negative control; however, log-phase growth resumes after 150 minutes. 
                            </h4>
                            <h4>CD27L endolysin results in decreased growth during the first 100 minutes, and growth thereafter appears to be linear. This may point to inhibited log-phase growth in subsequent generations of <i>B. subtilis</i> following CD27L exposure.</h4>

          </div></centre>
          </div>
          </div>
          </div>

          <h4>Our next step was then to titrate our killing assay with increasing concentrations of CD27L endolysin. This titration showed measurable effect on <i>B. subtilis</i> cultures in LB broth with final endolysin concentrations greater than 0.2mg/mL (Fig. 5).</h4>

          <h4>From this data, we were able to obtain kinetic constants for endolysin killing, which were subsequently used in our <ahref=”https://2019.igem.org/Team:Oxford/Model/Design>mathematical models</a>.</h4>
          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#CD27L_Fig5">Fig. 5: CD27L Endolysin Killing Assay Varying Final Endolysin Concentration</a>
                </h4>
              </div>
              <div id="CD27L_Fig5" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  
                  <img src="https://2019.igem.org/wiki/images/6/6b/T--Oxford--EndolysinTitration.png" style="width:100%" alt="FFig. 5: CD27L Endolysin Killing Assay Varying Final Endolysin Concentration">
               <h4>Here we show that the effect of the endolysin appears to inhibit growth at concentrations above 0.002mg/mL. Growth seems to increase significantly at 150 minutes, which may be indicative of CD27L degradation. </h4>
               <h4>Interestingly, 0.2mg/mL of endolysin grows linearly for duration of the assay (over 6 hours), as does 0.45mg/mL after 150 minutes. This may mean that log-phase growth is inhibited under these conditions.</h4>

</div></centre>
</div>
</div>
</div>

  <h4>We also showed that endolysin was specific to <i>B. subtilis</i> using a killing assay involving <i>E. coli</i>, <i>B. subtilis</i>, and our chassis <i>Lactobacillus reuteri</i> (Fig. 6 & 7).</h4>
  <br>
  <div class="panel-group">
      <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a data-toggle="collapse" href="#CD27L_Fig6">Fig. 6: CD27L Endolysin Killing Assay on <i>E. coli</i> versus <i>Bacillus subtilis</i></a>
        </h4>
      </div>
      <div id="CD27L_Fig6" class="panel-collapse collapse">
        <centre><div class="panel-body">
          
          <img src="https://2019.igem.org/wiki/images/e/e3/T--Oxford--EndolysinEcoli.png" style="width:100%" alt="Fig. 6: CD27L Endolysin Killing Assay on <i>E. coli</i> versus <i>Bacillus subtilis</i>">
          <h4>The above graph demonstrates the specificity of the CD27L endolysin, by assaying killing on <i>E. coli</i>, the most common chassis used in synthetic biology applications. </h4>

</div></centre>
</div>
</div>
</div>

  <div class="panel-group">
      <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a data-toggle="collapse" href="#CD27L_Fig7">Fig. 7: CD27L Endolysin Killing Assay on <i>Bacillus subtilis</i> versus <i>Lactobacillus reuteri</i></i></a>
        </h4>
      </div>
      <div id="CD27L_Fig7" class="panel-collapse collapse">
        <centre><div class="panel-body">
          
          <img src="https://2019.igem.org/wiki/images/7/73/T--Oxford--EndolysinLreuteri.png" style="width:100%" alt="Fig. 7: CD27L Endolysin Killing Assay on <i>Bacillus subtilis</i> versus <i>Lactobacillus reuteri</i>">
     
          <h4> While <i>L. reuteri</i> grows to a smaller maximum OD600 than <i>B. subtilis</i>, there is no discernable change in OD600 due to endolysin . Notably, the effect of lysozyme on <i>L. reuteri</i> is relatively small due to its thick bacterial cell wall, and we were unable to show a killing effect. For this reason, ampicillin was chosen as our positive control.</h4>
</div></centre>
</div>
</div>
</div>  

    <h4>Finally, we were fortunate enough to obtain scanning electron microscopy (SEM) images thanks to Errin Johnson at the Dunn School of Pathology. After performing our endolysin killing assay, we fixed the cells onto glass slides and imaged them. We chose this as a qualitative method to determine if CD27L had a similar effect on endolysin killing as lysozyme<sup>4</sup>. Below shows images of <i>B. subtilis</i> following CD27L endolysin treatment (Fig. 8). Here, we demonstrated some changes in <i>B. subtilis</i> cell morphology, but further investigation is needed to explore these effects.</h4>
    <br>

    <div class="panel-group">
        <div class="panel panel-default">
        <div class="panel-heading">
          <h4 class="panel-title">
            <a data-toggle="collapse" href="#CD27L_Fig8">Fig. 8: Scanning Electron Microscopy (SEM) Images of B. subtilis after 30 Minute Incubation with CD27L Endolysin versus Control.</i></a>
          </h4>
        </div>
        <div id="CD27L_Fig8" class="panel-collapse collapse">
          <centre><div class="panel-body">
            
            <img src="https://2019.igem.org/wiki/images/b/b5/T--Oxford--SEM-Bsubtilis-60min.png" style="width:100%" alt="Fig. 8: Scanning Electron Microscopy (SEM) Images of B. subtilis after 30 Minute Incubation with CD27L Endolysin versus Control.">
       
            <h4>Here, the SEM images show slightly deflated <i>B. subtilis</i> cells and cells with small bulges on them compared to the control. This may be due to compromised structure of the peptidoglycan scaffold and consequent bulging of the cell membrane. However, this will have to be explored further to definitively confirm the effects of CD27L endolysin on <i>B. subtilis</i> morphology.</h4>
  </div></centre>
  </div>
  </div>
  </div>  

  <h4>CD27L has been shown to work functionally against <i>B. subtilis</i> and expresses more stably than CD27L<sub>1-179</sub>. We have demonstrated its killing capacity both quantitatively and qualitatively, as well as its specificity for certain species of bacteria, though more work will be needed to prove its effect on <i>C. difficile</i> and its direct effects on colony morphology.</h4>

</div>
</div>
</div>
</section>

<section id="n2" style="background-color: #f2f2f2ff; padding-bottom:15px;">
  <div class="container">
    <div class="row">
      <div class=".col-md-8 .offset-md-2">
          <h2><i>Lactobacillus reuteri</i></h2>

          <h3>Transformation via Electroporation</h3>

          <h4>Electroporation is the standard method of transformation for <i>Lactobacillus</i> spp., with several well-established protocols existent in literature, such as the ones by Berthier<sup>5</sup>, Aukrust<sup>6</sup>, Spengler<sup>7</sup> etc. However, in spite of the previous success with such procedures, our attempts to transform the <i>L. reuteri</i> type strain, DSM20016, were never successful. </h4>

          <h4>As it was later revealed, in order for such a transformation to have worked, the plasmid DNA should have been isolated from <i>L. lactis</i>, rather than <i>E. coli</i>. This is because the <i>L. lactis</i>-derived plasmid methylation pattern prevents its digestion by the host bacterium.</h4>

          <h4>While amplification in <i>L. lactis</i> would not have been feasible at that stage in the project, we acquired a strain of <i>L. reuteri</i> 100-23c and a new transformation protocol (which can be found in our <a href=”https://2019.igem.org/Team:Oxford/Protocols>Protocols</a> page) from the Quadram Institute. From that point forward, our transformations succeeded. As proof, we performed colony PCR on transformants containing our control plasmid, pTRKH3-GFP, as well as the other constructs of CD27L (Figures 1, 3, 8).</h4>
        <br>
      
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_Fig1">Fig. 1: pTRKH3-GFP <i>L. reuteri</i> Transformants Colony PCR</a>
                </h4>
              </div>
              <div id="L_reuteri_Fig1" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  
                  <img src="https://2019.igem.org/wiki/images/b/b2/T--Oxford--pTRKH3_GFP_colony_PCR.png" style="width:100%" alt="Fig. 1: pTRKH3-GFP <i>L. reuteri</i> Transformants Colony PCR">
             
                  <p style="font-size: 14px;">Fig. 1: pTRKH3-GFP <i>L. reuteri</i> Transformants Colony PCR Conditions: Primers - S01 and S02 (our generic pTRKH3 sequencing primers). Cell lysate as sample template, and miniprepped vector as control template. Annealing temperature: 55 ⁰C, extension time: 2:00. NEB Taq polymerase 2x master mix.</p>
        </div></centre>
        </div>
        </div>
        </div>  
      
        <h4>Given the large number of protocols we had worked with, we believed it was appropriate to compare the transformation efficiency of two of them. As described in the <a href=”https://2019.igem.org/Team:Oxford/Measurements>Measurements</a> page, the transformation efficiency for the Quadram Institute protocol was on the order of 680 colony forming units per μg plasmid DNA. In contrast, the second protocol by Aukrust yielded no transformants.</h4>
      
          <h4>Additionally, the colony PCR reaction itself requires optimization, as its results were frequently inconsistent and often produced false negative results. The source of such errors was the difficulty in lysing the <i>L. reuteri</i> cells. As explained in our <a href="https://2019.igem.org/Team:Oxford/Protocols#Lactobacillus_colony_PCR"><i>Lactobacillus</i> colony PCR</a> protocol, this issue was circumvented by growing the colonies in liquid culture and then subjecting them to sonication, and finally using the lysate as the template for the PCR.</h4>
        
          </div>
          </div>
          </div>
        </section>

          <section id="n3" style="padding-bottom: 15px;">
              <div class="container">
                <div class="row">
                  <div class=".col-md-8 .offset-md-2">

          <h3>CD27L Endolysin</h3>

        <img src="https://2019.igem.org/wiki/images/1/1a/T--Oxford--slpMOD_CD27L.png" style="width:100%" alt="slpMOD_CD27L_Construct">
        <p style="font-size: 14px;">Fig. 2: SlpMod-CD27L: the diagram shows the organization of our construct as assembled in the pTRKH3 vector. The relevant features are the presence of the secretion (slpMod) and purification tags (6His and SpyTag) linked to the CD27L endolysin. Importantly, an mClover3 reporter is co-transcribed with CD27L, but not fused to it, in order to avoid altering the activity of the enzyme.
          </p>

          <h4>This section focuses on presenting the data acquired for CD27L in the context of <i>L. reuteri</i> expression. This is as opposed to its physicochemical properties and catalytic activity, which have been detailed above. </h4>
          <h4>Due to the obstacles encountered in the transformation of the strain and the relatively large number of false positives associated with it, it is appropriate to provide evidence of transformation, which was obtained by performing PCR on cellular lysate derived through the sonication of a liquid culture, as described in our <a href="https://2019.igem.org/Team:Oxford/Protocols#Lactobacillus_colony_PCR"><i>Lactobacillus</i> colony PCR</a> protocol and shown below in Figure 3.</h4>
          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_Fig3">Fig. 3: slpMod-CD27L <i>L. reuteri</i> Transformants Colony PCR</a>
                </h4>
              </div>
              <div id="L_reuteri_Fig3" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  
                  <img src="https://2019.igem.org/wiki/images/e/eb/T--Oxford--pTRKH3_CD27L_colony_PCR.png" style="width:100%" alt="Fig. 3: slpMod-CD27L <i>L. reuteri</i> Transformants Colony PCR">
             
                  <p style="font-size: 14px;">Fig. 3: slpMod-CD27L <i>L. reuteri</i> Transformants Colony PCR Conditions: Primers - S01 and S02 (our generic pTRKH3 sequencing primers). Cell lysate as sample template, and miniprepped vector as control template. Annealing temperature: 55 ⁰C, extension time: 2:00. NEB Taq polymerase 2x master mix.</p>
        </div></centre>
        </div>
        </div>
        </div>  

        <h4>Given the successful transformation of our constructs, our data could now be used to refine our mathematical models. Specifically, mathematical modelling of our system has shown that inducible expression of endolysin is worthwhile – as far as yield alone is concerned – only if protein expression causes cellular stress and significantly inhibits the growth of the chassis. </h4>
        
        <h4>To test whether cellular stress really occurs, we performed growth assays on the wild type strain and two of our transformed ones: pTRKH3 slpMod-CD27L and pTRKH3 erm-GFP in parallel. As expected, the transformed strains showed reduced growth rates. However, strikingly, the transformants containing CD27L and mClover (pTRKH3 slpMOD-CD27L) grew faster than the ones which had been transformed with GFP alone (pTRKH3 erm-GFP), as shown in Figure 4.</h4>
        <br>
        <div class="panel-group">
            <div class="panel panel-default">
            <div class="panel-heading">
              <h4 class="panel-title">
                <a data-toggle="collapse" href="#L_reuteri_Fig4">Fig. 4: <i>Lactobacillus reuteri</i> Growth Curves</a>
              </h4>
            </div>
            <div id="L_reuteri_Fig4" class="panel-collapse collapse">
              <centre><div class="panel-body">
                
                <img src="https://2019.igem.org/wiki/images/5/5a/T--Oxford--GrowthRateLreutTransformants.png" style="width:100%" alt="Fig. 4: <i>Lactobacillus reuteri</i>Growth Curves">
           
                <p style="font-size: 14px;">Fig. 4: <i>Lactobacillus reuteri</i> Growth Curves: Bacterial growth was measured as a function of optical density at 600 nm (OD600) using a 96-well plate reader. The cells were grown on MRS containing 5 μg/mL erythromycin, for the transformants, or no antibiotic for wild type. Antibiotic-free controls were set up to prove that the reduced growth rate is not due to the presence of Erythromycin; as expected, no statistically significant difference was observed between controls and erythromycin positive cultures (data not shown). Each data point is the average of 6 blank-corrected replicates. Total volume per well = 500 μL.</p>
      </div></centre>
      </div>
      </div>
      </div>  

        <h4>Based on the growth data, regressions curves were determined for the exponential growth phase using ANCOVA analysis and the doubling times of each of the strains calculated. This has revealed trends not readily perceivable by visual inspection of the graphs - specifically that the GFP strain’s growth rate was reduced by 66%, while that of the CD27L’s only by 19%. </h4>      

        <div class="table-responsive">
            <table class="table">
              <thead>
                <tr>
                  <th><i>L. reuteri</i> Strain</th>
                  <th>Doubling Time /minutes</th>
                  <th>R-Squared Value</th>
  
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>pTRKH3-Perm-CD27L-mClover3 L. reuteri 10023C (MRS)</td>
                  <td>39.2</td>
                  <td>0.971</td>
                </tr>

                <tr>
                  <td>pTRKH3-Perm-GFP L. reuteri 10023C (MRS)</td>
                  <td>54.5</td>
                  <td>0.963</td>
                </tr>

                <tr>
                  <td>Wild Type L. reuteri 10023C (MRS)</td>
                  <td>32.9</td>
                  <td>0.973</td>
                </tr>
                  
  
              </tbody>
            </table>
          </div>

          <h4>These results are surprising as expression of both the endolysin and the fluorescent reporter is expected to result in larger burden on the cell compared to the expression of reporter alone. </h4>

          <h4>The next logical step was to check whether any correlation exists between the levels of foreign protein expression and cell proliferation rate.. This was done by measuring the fluorescence of the strains as they were growing as shown in Figure 5.</h4>
          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_Fig5">Fig. 5: Transformant Fluorescence</a>
                </h4>
              </div>
              <div id="L_reuteri_Fig5" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  
                  <img src="https://2019.igem.org/wiki/images/6/6f/T--Oxford--CorrectedFI.png" style="width:100%" alt="Fig. 5: Transformant Fluorescence">
             
                  <p style="font-size: 14px;">Fig. 5: Transformant Fluorescence: Normalized fluorescence of the cell cultures was determined by calculating the ratio of raw fluorescence to the optical density at 600 nm to ensure that the fluorescence levels measured are a result of reporter gene expression and not simply due to cell growth. The data were further calibrated as described on the Measurements page. The observed drop in fluorescence/OD600 across the time period can be accounted for by considering the large background fluorescence signal of MRS, such that the fluorescence stays nearly constant, while the OD increases markedly as a result of cell proliferation.</p>
                  <h4>For the first few hours at OD600 of less than 0.2, the fluorometer was relatively imprecise with the OD600 fluctuating compared to previous measurements. This is why the graph above starts around the 4hr mark, where the lack of precision of the fluorometer would lead to a smaller uncertainty in its readings. </h4>
                </div></centre>
        </div>
        </div>
        </div> 

          <h4>The fluorescence data indicates that the GFP strain has the highest level of protein expression whereas the slpMod-CD27L one has slightly less expression initially but then reaches baseline after 9-10 hours from inoculation. </h4>
          <h4>Combining the scattering and fluorescence data suggests an inverse relationship between exogenous protein expression and growth rate. This is consistent with a model in which the <i>L. reuteri</i> strain downregulates protein expression to enable the redirection of its resources towards growth and division. </h4>
          <h4>Given the increased complexity of the CD27L construct, elevated cellular burden is to be expected, such that one could postulate the existence of a negative feedback loop whereby protein expression is reduced in response to cellular stress above a specific threshold value. </h4>
          <br>
          <h4>Further confirmation of the fluorometry assay results was obtained from fluorescence microscopy, a significantly more sensitive technique. As expected, the largest signal was observed for the GFP strain, and approximately none was measured for wild type. Interestingly, the CD27L strain showed little fluorescence and the mClover3 signal was concentrated in discrete spots inside the cell - the morphology of which is characteristic of inclusion bodies.</h4>
          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_Fig6">Fig. 6 Fluorescence Microscopy</a>
                </h4>
              </div>
              <div id="L_reuteri_Fig6" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  
                  <img src="https://2019.igem.org/wiki/images/d/d1/T--Oxford--Lactobacillus_microscopy.png" style="width:100%" alt="Fig. 6 Fluorescence Microscopy">
             
                  <p style="font-size: 14px;">Fig. 6 Fluorescence Microscopy: top row: micrographs of normalised exposure show the relative levels of exogenous protein expression in 3 strains of <i>Lactobacillus reuteri</i> 100-23c: wild type, pTRKH3-erm-GFP and pTRKH3-erm-slpMod CD27L_mClover. Bottom row: the corresponding bright field imaging mode. As expected, no fluorescent protein expression is detected in the wild type strain, while significant levels are observable in the GFP transformants. However, the CD27L shows low level expression concentrated in inclusion body-like structures.</p>
        </div></centre>
        </div>
        </div>
        </div>

        <h4>This observation strongly supports the hypothesis that rapid protein accumulation leads to expression downregulation, which in turn decreases the doubling time. This bears some comparison to the unfolded protein response in eukaryotes, and it might be interesting to probe the roles played by chaperones and/or approach the problem from a proteomics perspective to learn more about the global cellular response to stress. </h4>
        <h4>Such lines of future research may be particularly important in light of the fact that no endolysin expression could be detected by either western blotting with anti-6His antibodies, or by SpyCatcher band shift assays. </h4>
        <h4>A better understanding of the pathways involved in stress response might allow us to use synthetic biology to improve the resilience of the strain and allow it to yield therapeutically effective amounts of endolysin. Simple potential alternative solutions include designing a Lactobacillus-specific vector with an inducible promoter, or reducing the strength of the promoter. </h4>
      </div>

      </div>
      </div>
      </section>

      <section id="n4" style="background-color: #f2f2f2ff; padding-bottom:15px;">
          <div class="container">
            <div class="row">
              <div class=".col-md-8 .offset-md-2">

   
        <h3>AgrAC</h3>

        <img src="https://2019.igem.org/wiki/images/2/2e/T--Oxford--AgrAC.png" style="width:100%">
        <p style="font-size:14px;">Fig. 7: AgrAC The organization of the AgrAC construct in the pTRKH3 vectors. The main features include regulation of both genes by the weak slp promoter, resulting in a single mRNA containing two independent reading frames: AgrA-SpyTag and AgrC-HA</p>

        <h4>As explained previously, the AgrC gene encodes a membrane bound receptor with histidine kinase activity induced by the binding of the C. difficile autoinducer peptide (AIP). During activation, the receptor autophosphorylates one of its histidine residues and then transfers to moiety to an aspartate residue of AgrA. This induces AgrA’s dimerization and activation, such that the dimer can bind its cognate promoter and initiate transcription.</h4>
        <h4>Figure 8 below shows the successful transformation of the AgrA and AgrC constructs.</h4>
        <br>
        <div class="panel-group">
            <div class="panel panel-default">
            <div class="panel-heading">
              <h4 class="panel-title">
                <a data-toggle="collapse" href="#L_reuteri_Fig8">Fig. 8: AgrAC <i>L. reuteri</i> Transformants Colony PCR</a>
              </h4>
            </div>
            <div id="L_reuteri_Fig8" class="panel-collapse collapse">
              <centre><div class="panel-body">
                
                <img src=" https://2019.igem.org/wiki/images/e/e9/T--Oxford--pTRKH3_AgrAC.png" style="width:100%" alt="Fig. 8: AgrAC <i>L. reuteri</i> Transformants Colony PCR">
           
                <p style="font-size: 14px;">Fig. 8: AgrAC <i>L. reuteri</i> Transformants Colony PCR Conditions: Primers - S01 and S02 (our generic pTRKH3 sequencing primers). Cell lysate as sample template, and miniprepped vector as control template. Annealing temperature: 55 ⁰C, extension time: 2:30. NEB Taq polymerase 2x master mix.</p>
      
      
              </div></centre>
      </div>
      </div>
      </div>
      
        <h4>Given successful transformation, in order to test the system’s feasibility in <i>L. reuteri</i>, several experiments were planned, such as the construction of a reporter system that could be induced by AIP, or the detection of phosphorylated AgrA and the subsequent determination of signalling dynamics parameters. This would have enhanced the accuracy of our mathematical model. </h4>
        <h4>However, similarly to the situation of CD27L, no protein could be detected by either Anti-HA western blot or in-gel SpyCatcher detection. This is likely for similar reasons as mentioned previously.  As a potential solution, we attempted to grow our transformed cells at lower temperature (28 ⁰C instead of 37 ⁰C) in order to allow for increased protein folding opportunity, but no positive results were obtained. </h4>
        <h4>Another potential explanation for the lack of protein expression may be the very weak activity of the slp promoter in Lactobacillus species, as shown by Lizier et al. (2010)<sup>8</sup>. While slp had been chosen specifically to avoid overloading the cell with proteins not required at high concentrations, further optimization of the promoter and other regulatory sequences needs to be optimised. The need for improvement is further compounded by the fact that AgrAC must be expressed constitutively, unlike CD27L, in order to enable the detection of AIP.</h4>
      </div>
      </div>
    </div>
      </section>

      <section id="n5" style="padding-bottom: 15px;">
          <div class="container">
            <div class="row">
              <div class=".col-md-8 .offset-md-2">

        <h3> SpyTag</h3>

        <h4>SpyTag has previously been used as a platform for forming isopeptide bonds. We were able to demonstrate its ability to purify proteins for downstream applications. Thus, this shows how SpyTag can be used as a versatile single tag within a coding sequence.</h4>


        <h4>In our endolysin constructs, SpyTag was used as a secondary affinity tag. The purification method was developed by Anuar et al. (2019)<sup>9</sup>. It uses an immobilized protein, SpyDock, which can bind SpyTag reversibly. SpyDock captures fusion tags and allows elution at high (2.5M) imidazole concentration. </h4>

        <h4>To assess the capabilities for purification, we used densitometric analysis to determine how pure our protein product is and how the purification relates to the widely-used 6xHis Ni-NTA purification method. We used a construct that consisted of mClover3 with SpyTag and 6xHis tag (Fig 1).</h4>

        <img src="https://2019.igem.org/wiki/images/6/65/T--Oxford--Spy-mClover-His.png" style="width:75%; margin-left:auto; margin-right:auto; display: block;" alt="Construct_mClover3">
        <p style="font-size: 14px;">Fig. 1 Constructs</p>

        <h4>Purification was done in parallel using 250μL of Ni-NTA and Spy&Go packed resin. 10μg of total protein was loaded on the SDS-PAGE gels (Fig. 2 and 3) followed by comparison with densitometric analysis (Fig. 4). </h4>
        <br>

        <div class="panel-group">
            <div class="panel panel-default">
            <div class="panel-heading">
              <h4 class="panel-title">
                <a data-toggle="collapse" href="#SpyTag_Fig2">Fig. 2: Spy&Go Affinity Purification using SpyTag-mClover3-6His</a>
              </h4>
            </div>
            <div id="SpyTag_Fig2" class="panel-collapse collapse">
              <centre><div class="panel-body">
                
                <img src="https://parts.igem.org/wiki/images/b/bc/T--Oxford--SpyandGo.png" style="width:100%" alt="Fig. 2: Spy&Go Affinity Purification using SpyTag-mClover3-6Hiss">
           
      
              </div></centre>
            </div>
          </div>
          </div>

              <div class="panel-group">
                  <div class="panel panel-default">
                  <div class="panel-heading">
                    <h4 class="panel-title">
                      <a data-toggle="collapse" href="#SpyTag_Fig3">Fig. 3: Ni-NTA Affinity Purification using SpyTag-mClover3-6His</a>
                    </h4>
                  </div>
                  <div id="SpyTag_Fig3" class="panel-collapse collapse">
                    <centre><div class="panel-body">
                      
                      <img src="https://parts.igem.org/wiki/images/2/28/T--Oxford--NiNTA_Characterization.png" style="width:100%" alt="Fig. 3: Ni-NTA Affinity Purification using SpyTag-mClover3-6His">
                 
            
                    </div></centre>
                  </div>
                </div>
                </div>

                    
              <div class="panel-group">
                  <div class="panel panel-default">
                  <div class="panel-heading">
                    <h4 class="panel-title">
                      <a data-toggle="collapse" href="#SpyTag_Fig4">Fig. 4: mClover Purification Comparison Between Ni-NTA and Spy&Go Using Densitometry</a>
                    </h4>
                  </div>
                  <div id="SpyTag_Fig4" class="panel-collapse collapse">
                    <centre><div class="panel-body">
                      
                      <img src="https://parts.igem.org/wiki/images/4/4b/T--Oxford--Densitometry_Characterization2019.png" style="width:100%" alt="Fig. 4: mClover Purification Comparison Between Ni-NTA and Spy&Go Using Densitometry">
                      
            
                    </div></centre>
                  </div>
                </div>
                </div>

                    <h4>Spy&Go purification was able to purify the protein of interest with 60.1% (±4.4%, n=3) purity. This is lower than that of the Ni-NTA purification (74.5%, ±5.0%, n=3)</h4>
                    <h4>Using this data and the results of BCA assay, we were able to determine that the total concentration of mClover3 from the Spy&Go purification was 1.0 mg/ml (±0.316mg/ml, n=3), while the Ni-NTA purification yielded 1.5 mg/ml (±0.344mg/ml, n=3). </h4>
                    <h4>From this, we were able to calculate the approximate yield. Spy&Go purification yielded a 8.2 mg/L (±2.525, n=3) of mClover3, whereas Ni-NTA purification yielded a 15.4mg/L of mClover3 (±3.438, n=3).</h4>
                    <br>
                    <h4>Overall, this demonstrates SpyTag as a potential multi-use tag, now allowing for purification in addition to downstream SpyCatcher applications.</h4>

                     </div>
                     </div>
                    </div>
                     </section> 

                     <section id="n6" style="background-color: #f2f2f2ff; padding-bottom:15px;">
                        <div class="container">
                          <div class="row">
                            <div class=".col-md-8 .offset-md-2">

                    <h3>AIP (Auto-Inducing Peptide)</h3>

        <h4>To test the efficacy of our own future quorum sensing mechanism in <i>Lactobacillus reuteri</i>, we had to obtain or synthesise <i>C. difficile’s</i> auto-inducing peptide (AIP). iGEM Nottingham very kindly sent us samples of <i>C. difficile</i>’s supernatant to help us characterise <i>C. difficile</i>’s AIP. These three samples included sterile culture media, supernatant from a 3 hour pre-log phase culture and supernatant from a 48hr culture. </h4>
        <h4>Mass spectra were then obtained for serial dilutions of the three respective samples. Electron spray ionisation (ESI) was the specific technique used with the sterile media used as background for both the 3hr and 48hr samples.  The mass spectra were then compared to each other. </h4>
        <h4>The 3hr pre-log sample was also used as background against the 48hr, with the rationale being that the 48hr sample would have AIP in a much greater abundance. We hoped to confirm the conclusions made by Darkoh et al. (2014)<sup>8</sup> where he states that the molecular weight of the AIP is 612.37 Da. This would be consistent the cyclic peptide shown in (Fig. 1).</h4>

        <img src="https://2019.igem.org/wiki/images/2/2b/T--Oxford--AIP.png" style="width:75%; margin-left:auto; margin-right:auto; display: block;" alt="Cyclic_Peptide_1">

        <h4>Unfortunately, amidst all our spectra, only one yielded a small peak around 612.36 Da, as seen in Figure 2.1.</h4>
        <h4>Mass spectra for the 3hr prelog supernatant and 48hr supernatant samples were first obtained. These spectra were compared to determine the relative abundances of different peaks.
            AIP molecules are generally in the range of approximately 500 Da to 1100 Da so we focussed on the peaks within this range.
            </h4><br>

            <div class="panel-group">
                <div class="panel panel-default">
                <div class="panel-heading">
                  <h4 class="panel-title">
                    <a data-toggle="collapse" href="#AIP_Fig2">Fig. 2: Mass Spectra Sample Corrected Using Sterile Media, 600Da Range</a>
                  </h4>
                </div>
                <div id="AIP_Fig2" class="panel-collapse collapse">
                  <centre><div class="panel-body">

                    
                                <img src="https://2019.igem.org/wiki/images/f/ff/T--Oxford--MS1.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig 2.1: Mass Spectrometry 3-Hour Pre-Log Sample Corrected Using Sterile Media, 600Da Range">
                                <p style="font-size: 14px;">Fig 2.1: Mass Spectrometry 3-Hour Pre-Log Sample Corrected Using Sterile Media, 600Da Range</p>
                            <br>
                           
 
                                <img src="https://2019.igem.org/wiki/images/d/d5/T--Oxford--MS2.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig 2.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using Sterile Media, 600Da Range">
                                <p style="font-size: 14px;">Fig 2.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using Sterile Media, 600Da Range</p>
  
           
                          <h4> “Oh AIP, AIP! wherefore art thou AIP?”. Any of these peaks could be for the AIP, so such data is inconclusive.</h4>
                      </div></centre>
                    </div>
                    </div>
                    </div>

                    <h4>Upon comparison of the two mass spectra, it was evident that the 48hr sample generally had greater abundance of the molecular species present. However, it is also clear that the peaks at 1018.9,1030.9 and 1070.9 Da had greater abundance in the 48hr sample compared to the 3hr pre-log sample. This is in opposition to Darkoh’s original findings<sup>10</sup> which showed that toxin-induced activity ceased when the supernatant was passed through a 1kDa filter. 
                    </h4><br>

                    <div class="panel-group">
                        <div class="panel panel-default">
                        <div class="panel-heading">
                          <h4 class="panel-title">
                            <a data-toggle="collapse" href="#AIP_Fig3">Fig. 3: Mass Spectra Sample Corrected Using Sterile Media, 1000Da Range</a>
                          </h4>
                        </div>
                        <div id="AIP_Fig3" class="panel-collapse collapse">
                          <centre><div class="panel-body">
                        
                                        <img src="https://2019.igem.org/wiki/images/5/50/T--Oxford--MS3.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig. 3.1: Mass Spectrometry 3-Hour Pre-Log Sample Corrected Using Sterile Media, 1000Da Range">
                                        <p style="font-size: 14px;">Fig. 3.1: Mass Spectrometry 3-Hour Pre-Log Sample Corrected Using Sterile Media, 1000Da Range</p>
                                    <br>
                                    
                                    
                                        <img src="https://2019.igem.org/wiki/images/1/1a/T--Oxford--MS4.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig. 3.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using Sterile Media, 1000Da Range">
                                        <p style="font-size: 14px;">Fig. 3.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using Sterile Media, 1000Da Range</p>
          
                                
                                
                                  <h4> </h4>
                              </div></centre>
                            </div>
                            </div>
                            </div>

                            <h4>Further insight was provided when the 3hr pre-log spectra were used as background for the 48hr spectra.  This was also carried out for 48hr samples treated with hydroxylamine for comparison.</h4>
                            <h4>Hydroxylamine would attack the thioester bond, and in an addition-elimination type reaction should add around 33Da to the molecular ion peak, assuming that the thioester is part of the cyclic system. This is exemplified in Figure 1, again assuming Darkoh’s proposed structure.</h4>
                            <h4>Specifically, following acetone purification, some samples were incubated with 200 mM hydroxylamine at room temperature for two hours at a pH of 8.5.</h4>

                            <h4>Unfortunately, no noticeable shifts in large peaks were seen as shown in Figure 4. The large peaks around 1000Da are still present. However, the information gained from ESI is limited, and we are hoping to do tandem mass spectrometry (MS/MS) on the peaks at 1018.9, 1030.9 and 1070.9 Da to see the fragmentation patterns and hopefully discern the structures of these molecules. </h4>
                            <br>

                            <div class="panel-group">
                                <div class="panel panel-default">
                                <div class="panel-heading">
                                  <h4 class="panel-title">
                                    <a data-toggle="collapse" href="#AIP_Fig4">Fig. 4: Mass Spectra Sample Corrected Using 3-Hour Pre-Log Sample, Pre/Post-Hydroxylamine Treatment, 600Da Range</a>
                                  </h4>
                                </div>
                                <div id="AIP_Fig4" class="panel-collapse collapse">
                                  <centre><div class="panel-body">
                                 
                                                <img src="https://2019.igem.org/wiki/images/5/57/T--Oxford--MS5.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig. 4.1: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 600Da Range">
                                                <p style="font-size: 14px;">Fig. 4.1: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 600Da Range</p>
                                            <br>
                                            
                                        
                                                <img src="https://2019.igem.org/wiki/images/7/73/T--Oxford--MS6.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig. 4.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 600Da Range, Post-Hydroxylamine Treatment">
                                                <p style="font-size: 14px;">Fig. 4.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 600Da Range, Post-Hydroxylamine Treatment</p>
                  
                           
                                        
                                          <h4> Peaks affected by hydroxylamine in the manner described previously should shift 33Da upwards. However, none of the peaks seem to shift in this manner. This may indicate the absence of a thioester or, more likely, that none of these peaks constitute the AIP.</h4>
                                      </div></centre>
                                    </div>
                                    </div>
                                    </div>

                                    <div class="panel-group">
                                        <div class="panel panel-default">
                                        <div class="panel-heading">
                                          <h4 class="panel-title">
                                            <a data-toggle="collapse" href="#AIP_Fig5">Fig. 5: Mass Spectra Sample Corrected Using 3-Hour Pre-Log Sample, Pre/Post-Hydroxylamine Treatment, 1000Da Range</a>
                                          </h4>
                                        </div>
                                        <div id="AIP_Fig5" class="panel-collapse collapse">
                                          <centre><div class="panel-body">
                                               
                                                        <img src="https://2019.igem.org/wiki/images/6/6f/T--Oxford--MS7.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig. 5.1: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 1000Da Range">
                                                        <p style="font-size: 14px;">Fig. 5.1: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 1000Da Range</p>
                                                    <br>
                                                   
                                               
                                                        <img src="https://2019.igem.org/wiki/images/9/9f/T--Oxford--MS8.png" style="width: 75%; display: block; margin-left:auto; margin-right: auto;" alt="Fig. 5.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 1000Da Range Post-Hydroxylamine Treatment">
                                                        <p style="font-size: 14px;">Fig. 5.2: Mass Spectrometry 48-Hour Stationary Sample Corrected Using 3-Hour Pre-Log Sample, 1000Da Range Post-Hydroxylamine Treatment</p>
                          
                                 
                                                
                                                  <h4>Peaks affected by hydroxylamine in the manner described previously should shift 33Da upwards.  As in Fig 4, it doesn't appear that any of the peaks shift in this manner. Significant noise is also present in this sample, as the abundance of the peaks is much weaker; however, given that the peaks have not changed, little can be concluded from these data.</h4>
                                              </div></centre>
                                            </div>
                                            </div>
                                            </div>

                                            <h4>In our search for the AIP, we certainly made headway towards discovering its chemical identity. However, we were not able to confirm the information stated previously by Darkoh et al. (2014)<sup>10</sup> We anticipate that tandem mass-spectrometry may be critical in determining AIP’s true identity.</h4>

    </div>
  </div>
  </div>
</section>

<section id="n7">>
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">

            <h2 style="text-align: center;">
              References
              </h2>

            <div class="table-responsive">
              <table class="table">
                <thead>
                  <tr>
                    <th> #</th>
                    <th>Reference</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>1</td>
                    <td>Dunne, Matthew et al. “The CD27L and CTP1L endolysins targeting Clostridia contain a built-in trigger and release factor.” PLoS pathogens vol. 10,7 e1004228. 24 Jul. 2014, doi:10.1371/journal.ppat.1004228</td>
                      
                  </tr>

                  <tr>
                    <td>2</td>
                    <td>Schleifer, K H, and O Kandler. “Peptidoglycan types of bacterial cell walls and their taxonomic implications.” Bacteriological reviews vol. 36,4 (1972): 407-77.</td>
                  </tr>

                    <tr>
                      <td>3</td>
                      <td>Mayer, Melinda J et al. “Structure-based modification of a Clostridium difficile-targeting endolysin affects activity and host range.” Journal of bacteriology vol. 193,19 (2011): 5477-86. doi:10.1128/JB.00439-11</td>
                    </tr>

                    <tr>
                      <td>4</td>
                      <td>Twetman, Svante, et al. “Scanning Electron Microscopic Study of Streptococcus Mutans BHT Lysed by Lysozyme.” European Journal of Oral Sciences, vol. 93, no. 1, Feb. 1985, pp. 23–29., doi:10.1111/j.1600-0722.1985.tb01304.x.</td>
                    </tr>

                    <tr>
                    <td>5</td>
                    <td>Berthier F. Zagorec M. Champomier-Vergès M. Ehrlich S.D. Morel-Deville F. (1996) Efficient transformation of Lactobacillus sake by electroporation. Microbiology 142, 1273–1279.</td>
                  </tr>

                  <tr>
                    <td>6</td>
                    <td>Aukrust, T. W., M. B. Brurberg, and I. F. Nes. 1995. Transformation of Lactobacillus by electroporation. Methods Mol. Biol. 47:201-208</td>
                      
                  </tr>
                  
                  <tr>
                    <td>7</td>
                    <td>Joseph R Spangler, Julie C Caruana, Daniel A Phillips, Scott A Walper, Broad range shuttle vector construction and promoter evaluation for the use of Lactobacillus plantarum WCFS1 as a microbial engineering platform, Synthetic Biology, Volume 4, Issue 1, 2019, ysz012,</td>
                  </tr>

                  <tr>
                    <td>8</td>
                    <td>Lizier, Michela, et al. “Comparison of Expression Vectors in Lactobacillus Reuteri  Strains.” FEMS Microbiology Letters, vol. 308, no. 1, Aug. 2010, pp. 8–15., doi:10.1111/j.1574-6968.2010.01978.x.</td>
                  </tr>

                  <tr>
                    <td>9</td>
                    <td>Anuar, I. N. A. K., Banerjee, A., Keeble, A. H., Carella, A., Nikov, G. I., & Howarth, M. (2019). Spy&Go purification of SpyTag-proteins using pseudo-SpyCatcher to access an oligomerization toolbox. Nature Communications, 10(1), 1–13. https://doi.org/10.1038/s41467-019-09678-w</td>
                  </tr>

                  <tr>
                    <td>10</td>
                    <td>Darkoh, Charles, et al. “Toxin Synthesis by Clostridium Difficile Is Stringently Regulated Through Quorum Signalling.” American Society for Microbiology, Cell-Cell Communication in Bacteria, 2014, pp. 84–84.</td>
                  </tr>

                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </section>

<style>
    .column {
  float: left;
  width: 25%;
  padding: 5px;
}
</style>

</body>
</html>
{{Oxford/Footer}}
