{{Oxford/Loader}}
{{Oxford}}
<html lang="en">

<!-- THIS IS WHERE THE HTML BEGINS -->

<body>
  
    <section id="banner">
        <div class="container" style="width:100%; padding-right:0; padding-left:0;">
          <div class="row">
            <div class="col-md-12">
          <div class="inner" style="background-image: url('https://2019.igem.org/wiki/images/6/6b/T--Oxford--Sponsors_Bear.jpeg'); background-size: cover;">
            <div class=image_faded_overlay" style="background-color: lightskyblue; opacity:0.2">
            <div class="translucent-back" style="height: 400px;">
            </div>
          </div>
          <div class="banner_overlay_text">
            Future Experiments
          </div>
          </div>
  
        </div>
      </div>
    </div>
        </section>

      <section id="n1" style="padding-bottom: 15px;">
        <div class="container">
          <div class="row">
            <div class=".col-md-8 .offset-md-2">

              <h2>CD27L Endolysin Testing</h2>

              <h4>The next logical step to determine its effectiveness would be confirming CD27L’s <i>C. difficile</i> lytic activity. This would also be important in obtaining more realistic rate constants for CD27L’s activity. Moreover, testing the effect of CD27L<sub>1-179</sub> as well as CD27L would be important too, as the truncation may prove exceptionally effective on <i>C. difficile</i>, despite results showing otherwise in <i>B. subtilis</i>.
              </h4>

              <h4>Moreover, we would have liked to oligomerise CD27L using previously described methods<sup>1</sup>. Potentially, the genes for tetramerisation of CD27L could be implemented <i>in vivo</i> to improve killing activity. This could easily be tested in our killing assay,</h4>

              <h4>The specificity of the CD27L endolysin could also be tested on many species of gut bacteria using our killing assay. Ensuring that CD27L doesn’t drastically alter the gut microbiome is key to our ethos, and confirming the specificity shown in Mayer et al. (2011)<sup>2</sup> in a similar environment to the gut would further prove this endolysin’s potential.</h4>

              <h4>Finally, the activity of CD27L should be assayed against <i>C. difficile</i> itself, rather than our safer <i>B. subtilis</i> model. This  is to ensure that all our results are transferable to a scenario as realistic as possible. In order to achieve this, more realistic bacterial concentrations should be assayed against via the aid of more sensitive live bacteria fluorescent reporters.</h4>
</div>
</div>
</div>
</section>

<section id="n2" style="background-color: #f2f2f2ff; padding-bottom: 15px;">
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">
          <h2><i>Lactobacillus reuteri</i>  CD27L Expression and Secretion</h2>

          <h4> As explained on the <a href=”https://2019.igem.org/Team:Oxford/Results”>Results</a> page, the main issue we have encountered with our work on CD27L in <i>L. reuteri</i> was actual expression of CD27L endolysin. Our hypothesis for such observed behaviour was that constitutive endolysin expression resulted in excessive cellular stress, which was mitigated through the downregulation of expression and proteolysis. </h4>

          <h4>Consequently, the main aim of our future experiments will be to gain a better understanding of the processes that govern the bacterial unfolded protein response, potentially through a proteomics approach in determining the global cellular response to stress, or through genetics experiments on the role of chaperones. </h4>

          <h4>Further testing will involve the adjustment of the parameters of the regulatory elements such that our system approximates  the ideal scenario described by our computational model. The secretion efficiency of the slpMod secretion tag linked to CD27L should also be determined, through quantification of relative protein concentration in the cytoplasm and extracellular space. </h4>
          
          <h4>In essence, it must be ascertained that, through the control of gene expression and secretion, our chassis can release sufficient endolysin in the extracellular space to limit the growth of <i>C. difficile</i> to a therapeutically effective degree.</h4>

          </div>
          </div>
          </div>
</section>

<section id="n3" style="padding-bottom: 15px;">
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">

          <h2>AgrAC Sensing System</h2>

          <h4>The AgrAC two-component system is a critical part of our design which we were unable to fully characterise. The system is especially difficult to isolate, as AgrC is a membrane protein. Detecting this system would involve identifying expression. We attempted this with SpyTag-SpyCatcher gel-shift assays as described <a href=”http://parts.igem.org/Part:BBa_K3183102”>here.</a></h4>

          <h4>Future work would ideally prove that phosphorylation occurs in this two-component system in a similar manner to <i>Staphylococcus aureus</i><sup>3</sup> via 2D gel electrophoresis. Dimerisation could also be proven using dynamic light scattering (DLS)/multi-angle light-scattering (MALS) in vitro, or cross-linking in vivo. </h4>

          <h4>The other key piece to the AgrAC sensing system is finding the putative promoter. Phosphorylated and/or dimerised AgrA could be fixed to a column using a tag (e.g. SpyTag) and DNA pull-downs could be performed following genomic DNA shearing. This could show potential DNA binding regions which haven’t been proven from previous gene-expression experiments<sup>4</sup>. Further confirmation could be obtained from footprinting, electrophoretic mobility shift assays (EMSA), or nitrocellulose filter assays.></h4>

          <h4>Ultimately, the goal is to show that a fully functional signalling system can be integrated in the cell biology of <i>Lactobacillus</i> and that expression of the endolysin can be induced by the binding of <i>C. difficile</i> AIP to the AgrC receptor. This could be tested using a fluorescent reporter assay. </h4>


        </div>
</div>
</div>
</section>

<section id="n4" style="background-color: #f2f2f2ff; padding-bottom: 15px;">
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">
          <h2>Autoinducer Peptide (AIP)</h2>

          <h4>Autoinducer peptide detection (AIP) would prove useful to testing our AgrAC sensing system. Moreover, it would provide a key piece of knowledge towards elucidating the quorum sensing pathway in <i>C. difficile</i>. Progressing from our electrospray ionisation mass spectrometry (ESI-MS), tandem mass spectrometry could determine whether certain peaks constitute part of the AgrD pre-peptide. Moreover, recombinant synthesis of AgrBD has been shown in <i>E. coli</i>, which could be used for further testing downstream of the AgrAC system<sup>5</sup>.</h4>

          </div>
          </div>
          </div>
</section>

<section id="n5" style="padding-bottom: 15px;">
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">

          <h2>Final Construct & Modelling Insights</h2>

          <h4>The final construct (Fig. 1) would allow for constitutive AgrA and AgrC expression, as well as CD27L expression under the control of a well-characterised Agr promoter. This would allow <i>L. reuteri</i> to employ the sensing system required for the expression of CD27L in response to <i>C. difficie</i>. Proving the functionality of this construct in <i>L. reuteri</i> is necessary to finalise a proof of concep</h4>
          
          <h4>Ultimate confirmation of our system’s feasibility would be to prove the inhibition of <i>C. difficile</i> growth in vitro in the presence of the ProQuorum bacteria.</h4>
          
          <img src="" style="width:75%; margin-left:auto; margin-right: auto;" alt="Final Construct">

          <p style="font-size: 14px;">Fig. 1: Final Construct in Transformed <i>L. reuteri</i></p>

          <h4>To improve safety, this construct would subsequently be integrated into the genome. Discussions with bioethicist. Frances Butcher, and the CEO of ZBiotics, Zachary Abbott, helped determine that this would be the safest way to incorporate our system into bacteria destined for outside the lab, as explained in our <a href=”https://2019.igem.org/Team:Oxford/Human_Practices”> Human Practices</a> page. </h4>
          
          <h4>Furthermore, as explained on our <a href=”https://2019.igem.org/Team:Oxford/Safety”>Safety</a> page, the appropriate endogenous gene knock-outs could be performed to limit the spread of the transgenic <i>L. reuteri</i> into the environment. </h4>
        
          <h4>Moreover, insights from our modelling have shown that an additional secondary amplification system may improve transformed <i>L. reuteri</i>’s response to the AIP. Testing the efficacy of a secondary amplification system would also be needed to determine the best way to propagate a signal throughout a population of <i>L. reuteri</i>.</h4>

        


        </div>
</div>
</div>
</section>

<section id="n6" style="background-color: #f2f2f2ff; padding-bottom: 15px;">
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">
          <h2>Proof of Concept & Final Development</h2>

          <h4>Taking our proof of concept to market is a long process, compounded with the fact that genetically modified therapeutics are currently not permitted by either the Federal Drug Agency (FDA) or the European Medicines Agency (EMA). Below, we propose a pipeline of the necessary experiments which would be required, given the necessary policy changes, to bring our therapeutic to market (Fig. 2).</h4>
          <br>
          <img src="https://2019.igem.org/wiki/images/e/e2/T--Oxford--Pipeline.png" style="display: block; width:75%; margin-left: auto; margin-right: auto;" alt="Pipeline Diagram">

          

          </div>
          </div>
          </div>
</section>

<section id="n7">>
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">

            <h2 style="text-align: center;">
              References
              </h2>

            <div class="table-responsive">
              <table class="table">
                <thead>
                  <tr>
                    <th> #</th>
                    <th>Reference</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>1</td>
                    <td>Khairil Anuar, Irsyad N A et al. “Spy&Go purification of SpyTag-proteins using pseudo-SpyCatcher to access an oligomerization toolbox.” Nature communications vol. 10,1 1734. 15 Apr. 2019, doi:10.1038/s41467-019-09678-w</td>
                      
                  </tr>

                  <tr>
                    <td>2</td>
                    <td>Mayer, M. J., et al. “Structure-Based Modification of a Clostridium Difficile-Targeting Endolysin Affects Activity and Host Range.” Journal of Bacteriology, vol. 193, no. 19, 2011, pp. 5477–5486., doi:10.1128/jb.00439-11.</td>
                  </tr>
                  <tr>
                    <td>3</td>
                    <td>Le, Katherine Y, and Michael Otto. “Quorum-sensing regulation in staphylococci-an overview.” Frontiers in microbiology vol. 6 1174. 27 Oct. 2015, doi:10.3389/fmicb.2015.01174</td>
                    </tr>

                    <tr>
                      <td>4</td>
                      <td>Essigmann, Heather T et al. “The Clostridium difficile quorum-sensing molecule alters the Staphylococcus aureus toxin expression profile.” International journal of antimicrobial agents vol. 49,3 (2017): 391-393. doi:10.1016/j.ijantimicag.2017.01.001</td>
                    </tr>

                    <tr>
                      <td>5</td>
                      <td>Thoendel, Matthew, and Alexander R Horswill. “Identification of Staphylococcus aureus AgrD residues required for autoinducing peptide biosynthesis.” The Journal of biological chemistry vol. 284,33 (2009): 21828-38. doi:10.1074/jbc.M109.031757 </td>
                    </tr>

                  

                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </section>


<style>
    .column {
  float: left;
  width: 25%;
  padding: 5px;
}
</style>

</body>
</html>
{{Oxford/Footer}}
