{{Oxford/Loader}}
{{Oxford}}
<html lang="en">

<!-- THIS IS WHERE THE HTML BEGINS -->

<body>
    <section id="banner">
        <div class="container" style="width:100%; padding-right:0; padding-left:0;">
          <div class="row">
            <div class="col-md-12">
          <div class="inner" style="background-image: url('https://2019.igem.org/wiki/images/0/0a/T--Oxford--Modleling_Filler2.jpg'); background-size: cover; background-position:center;">
            <div class="image_faded_overlay" style="background-color: white; opacity: 0.7">
              <div class="translucent-back" style="height: 400px;">
              </div>
            </div>
          <div class="banner_overlay_text">
            Modelling Design
          </div>
          </div>
            </div>
          </div>
        </div>
      </section>

      <section id="n1" style="padding-bottom:15px;">
        <div class="container">
          <div class="row">
            <div class=".col-md-8 .offset-md-2">
  
              <h2 style="text-align:center;"> Part I: Dynamic Expression vs. Constitutive Expression of Endolysin</h2>

              <h4>
                  To build our genetic circuit in vitro, we first needed to establish whether expression of endolysin in the ProQuorum system would be constitutive or regulated.
              </h4>
              <h4>
                  To inform our decision, we decided to build mathematical models of both forms of expression using some kinetic data constants derived from our wet lab work.
              </h4>

              <h4>
                Notably, the following kinetic constants were derived:
              </h4>

              <h4>
                a) Concentration of endolysin produced per <i>L. reuteri</i> cell required for lysis of <i>C. difficile</i> cells estimated from experimental killing assay data of surrogate target bacterium <i>B. subtilis</i> (further detailed in the <a href="https://2019.igem.org/Team:Oxford/Model/Supplementary_Materials">Supplementary Materials</a> page.)
              </h4>


                          <div class="table-responsive">
                            <table class="table">
                              <thead>
                                <tr>
                                  <th> Endolysin Concentration Produced per <i>L. reuteri</i> cell /M CFU <sup>-1</sup></th>
                                </tr>
                              </thead>
                              <tbody>
                                <tr>
                                  <td>0.00465</td>
                                </tr>
                              </tbody>
                            </table>
                          </div>

                      <h4>
                          b) Stress on <i>L. reuteri</i> growth rate due to constitutive expression of CD27L endolysin:
                      </h4>

                      
                      <div class="table-responsive">
                          <table class="table">
                            <thead>
                              <tr>
                                <th>Type of <i>L. reuteri</i></th>
                                <th>Doubling time /hr</th>
                                <th>Growth Rate (ln 2/Doubling Time) /hr <sup>-1</sup></th>
                              </tr>
                            </thead>
                            <tbody>
                              <tr>
                                <td>Wild Type</td>
                                <td>0.548</td>
                                <td>1.26</td>
                              </tr>
    
                              <tr>
                                <td>Consitutively Expressing CD27L Endolysin</td>
                                <td>9.08</td>
                                <td>0.76</td>
                              </tr>
                            </tbody>
                          </table>
                        </div>

                      <h4>
                          As such, the above kinetic data allows us to build the comparative mathematical method as mentioned above.
                      </h4><br>

                      </div>
                      </div>
                      </div>
      </section>

                      <section id="n2" style="background-color: #f2f2f2ff; padding-bottom:15px;">
                          <div class="container">
                            <div class="row">
                              <div class=".col-md-8 .offset-md-2">

                                <h3>Assumptions</h3>
                                <ul style="font-size: 18px;">
                                    <li>Each <i>L. reuteri</i> cell produces the theoretically predicted concentration of endolysin.</li>
                                    <li>When endolysin production is induced in the dynamic expression system, all cells begin synthesis simultaneously.</li>
                                    <li>Stress on <i>L. reuteri</i> cells due to expression of sufficient endolysin for lysis of <i>C. difficile</i> approximates to a decrease in growth rate from 1.26 hr<sup>-1</sup> to 1.06 hr<sup>-1</sup></li>
                                  </ul>

                                  <br>
                                  <div class="panel-group">
                                      <div class="panel panel-default">
                                      <div class="panel-heading">
                                        <h4 class="panel-title">
                                          <a data-toggle="collapse" href="#Diff_eq">Differential Equations</a>
                                        </h4>
                                      </div>
                                      <div id="Diff_eq" class="panel-collapse collapse">
                                        <centre><div class="panel-body">
        
                                            <div class="table-responsive">
                                                <table class="table">
                                                  <thead>
                                                    <tr>
                                                      <th>Process Modelled</th>
                                                      <th>Differential Equation 1</th>
                                                      <th>Explanation 1</th>
                                                      <th>Differential Equation 2</th>
                                                      <th>Explanation 2</th>
                                                    </tr>
                                                  </thead>
                                                  <tbody>
                                                    <tr>
                                                      <td>Logistic Growth Model of <i>L. reuteri</i> - Constitutive Expression</td>
                                                      <td>$$\frac {dC}{dt} = r_{stressed} \times C \times (1- \frac {C}{N_{lr}})$$</td>
                                                      <td>Endolysin is being expressed constitutively.</td>
                                                      <td>N/A</td>
                                                      <td>N/A</td>
                                                    </tr>
        
                                                    <tr>
                                                      <td>Logistic Growth Model of <i>L. reuteri</i> - Dynamic Expression</td>
                                                      <td>$$\frac {dR1}{dt} = r_{wildtype} \times R \times (1- \frac {R}{N_{lr}})$$</td>
                                                      <td>If endolysin is not being expressed.</td>
                                                      <td>$$\frac {dR2}{dt} = r_{stressed} \times R \times (1- \frac {R}{N_{lr}})$$</td>
                                                      <td>If endolysin is being expressed.</td>
                                                    </tr>
        
                                                    <tr>
                                                      <td>Linear Expression of Endolysin</td>
                                                      <td>$$\frac {dEc}{dt} = k \times C$$</td>
                                                      <td>For constitutive expression by <i>L. reuteri</i>
                                                      <td>$$\frac {dEr}{dt} = k \times R$$</td>
                                                      <td>For dynamic expression by <i>L. reuteri</i>
                                                      
                                                    </tr>
                                                  </tbody>
                                                </table>
                                              </div>
        

                                  </div>
                                  </div>
                                  </div>
                                </section>
                    
                                  <section id="n3" style="padding-bottom:15px;">
                                     

                                     

                                      <div class="container">

                                     

                                        <div class="row">
                                          <div class=".col-md-8 .offset-md-2">
                               
                                             <h3>Results</h3>
                                            

                                             <h4>Legend</h4>
                                               
                                             <div class="table-responsive">
                                                 <table class="table">
                                                   <thead>
                                                     <tr>
                                                       <th>Colour of Graph Line</th>
                                                       <th>Explanation</th>
                                                 
                                                     </tr>
                                                   </thead>
                                                   <tbody>
                                                     <tr>
                                                       <td>Dotted</td>
                                                       <td><i>L. reuteri</i> constitutively expressing endolysin.</td>
                                                     </tr>
   
                                                     <tr>
                                                       <td>Green</td>
                                                       <td>Endolysin yield increases with the time of induction of endolysin expression</td>
                                                     </tr>
   
                                                     <tr>
                                                       <td>Blue</td>
                                                       <td>Endolysin yield decreases with the time of induction of endolysin expression</td>
                                                     </tr>
   
                                                     <tr>
                                                       <td>Red</td>
                                                       <td>Endolysin yield of note is below endolysin yield from constitutive expression</td>
                                                     </tr>
                                                   </tbody>
                                                 </table>
                                               </div>
                                             </div>  
                                             <br>
                
                                                      <h4>To compare the outcomes of inducing endolysin at different time points to constitutive production, we plotted endolysin yields for <i> L. reuteri </i> constitutively expressing endolysin and <i> L. reuteri </i> regulating endolysin expression. The time points at which endolysin expression was induced in <i> L. reuteri </i> ranged between 2 hours and 32 hours. </h4>
                 
                                                      
                
                                                           
                
                                                                      
                                                                          <img src="https://2019.igem.org/wiki/images/1/13/T--oxford--constitutive_green.png" style="width:100%;" alt="Fig.   a: Endolysin Yield - Constitutive Expression & Varying Induction Time">
                                                                        
                                                                        <p style="font-size:14px;">Fig. 1a compares the yield of endolysin from constitutive expression to that from dynamic expression whereby yield <b>increases</b> with <b>increase</b> in the time of induction. </p>
                                                                        <h4>For induction times ranging from 2 hours to 16 hours, the endolysin yield is greater than the endolysin yield from constitutive production. </h4>
                                                                        <h4>In addition, the yield increases with increase in time point at which expression was induced. In the best case scenario, endolysin yield from induced expression at 16 hours is 1.5 times the endolysin yield from constitutive production.  </h4> 
                                                                        <h4>After the 16 hour time point, endolysin yields decrease with increase in the induction time. This is seen in Figure 1b below.</h4> 
                
                                                                        <br>
                                                                        
                                                                        <img src="https://2019.igem.org/wiki/images/0/01/T--oxford--constitutive_blueandred.png" style="width:100%;" alt="Fig.  b: Endolysin Yield - Constitutive Expression & Varying Induction Time">
                                                                            
                                                                        <p style="font-size:14px;">Fig. 2a compares the yield of endolysin from constitutive expression to that from dynamic expression whereby yield <b>decreases</b> with <b>increase</b> in the time of induction. </p>
                                                                        <h4>The graph shows that even though endolysin yields from dynamic expression are higher than those from constitutive expression, the yield decreases when the induction time is increased beyond 16 hours.</h4>
                                                                        <h4>After the 28 hour time point, the yield is lower than the yield from constitutive expression and continues to decrease as we increase the induction time.</h4>
                                                                        
                                                
                
                                                                        <br>
                
                                                                     
                
                                                                                <h4>To analyse the survivability of  <i>L. reuteri</i>  dynamically expressing endolysin to <i>L. reuteri</i> constitutively expressing endolysin, we plotted the growth curves corresponding to the endolysin yields above. The results are shown in Figure 2 below.</h4>
                                                                              
                                                                                  <img src="https://2019.igem.org/wiki/images/8/80/T--oxford--constitutivegrowthchart.png" style="width:100%;" alt="Fig. 3: Growth Curves of <i>L. reuteri</i>Constitutive Expression">
                
                                                                                  <p style="font-size:14px;">Fig 2: Growth Curves of  <i>L. reuteri</i> Undergoing Endolysin Expression with Induction Times ranging from 2 hrs to 23 hrs.</p>
                
                                                                                  <h4>The graph clearly depicts faster growth for the <i>L. reuteri</i> undergoing dynamic expression of endolysin. In addition to that, the more endolysin expression is delayed (i.e the greater the induction time), the faster the population of <i> L. reuteri </i> reaches its carrying capacity. </h4>
                                                                                  <br>
                                                                              
                                           
                                                                   
                                    
                                                                                     
                                                                                        
                                                                      
                
                
                                               
                                                                                        <h3>Discussion</h3>
                                               
                
                                                          <h4>From the results above, regulated production has advantages over constitutive production in:
                                                            </h4>
                
                                                            <ol style="font-size:18px;">
                                                              <li>Endolysin Yield
                                                                <br>If endolysin expression is induced before 18 hours, dynamic endolysin expression results in greater endolysin yields from <i>L. reuteri</i> than constitutive expression. 
                                                              </li>
                                                              
                                                              <br>
                                                              <li>Survivability of <i>L. reuteri</i>
                                                              <br>Dynamic endolysin expression reduces cell stress on <i>L. reuteri</i> and substantially increases survivability.. This is especially beneficial to our probiotic as it improves the chances of <i>L. reuteri</i> establishing itself in the competitive gut microbiome.
                                                            </li>
                                                             
                                                            </ol>
                
                                                            <h4>We therefore decided to choose a dynamic expression system, as opposed to a constitutive one,  in our probiotic. </h4>
                                                        
                                                            <br>
                                                   
                        
                                  </div></centre>
                              </div>
                              </div>
                              </div>

                            </div>
                          </div>
                        </div>
                            </section>

<section id="n2" style="background-color: #f2f2f2ff; padding-bottom:15px;">
  <div class="container">
    <div class="row">
      <div class=".col-md-8 .offset-md-2">
      
          <h2 style="text-align:center;">Part II: Mathematical Model of Two-Component Regulatory System (TCS)</h2>

          <h4>Once we established that expression of the endolysin in our <i>L. reuteri</i> should be regulated, we needed an external stimulus indicative of <i>C. difficile</i> infection and a regulatory system that would transduce such a stimulus to induce endolysin expression.
          </h4> <br>

          <div class="panel-group">
            <div class="panel panel-default">
            <div class="panel-heading">
              <h4 class="panel-title">
                <a data-toggle="collapse" href="#Stim_Req">External Stimulus Requirements</a>
              </h4>
            </div>
            <div id="Stim_Req" class="panel-collapse collapse">
              <centre><div class="panel-body">
                <h4>Given that our aim is to induce expression of endolysin upon detection of pathogenic <i>C. difficile</i> (as opposed to dormant colonies in the gut), the external signal must be a biomarker of the pathogen. Thus, an excellent candidate is the autoinducer peptide (AIP) molecule, a quorum sensing molecule produced by <i>C. difficile</i>  to trigger toxin expression in neighbouring cells.
                </h4>
                </div></centre>
            </div>
            </div>
            </div>

            <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#Reg_Req">Regulatory System Requirements</a>
                </h4>
              </div>
              <div id="Reg_Req" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  <h4>
                    For optimal function of our overall system, the regulatory system must meet certain requirements:
                  </h4>
        
                  <ul style="font-size: 18px;">
                    <li>Endolysin expression should be triggered when the surrounding population of <i>C. difficile</i> crosses a certain threshold.</li>
                    <li>The regulatory system must be robust to temporary spikes of external signal(s).</li>
                    <li>The timing and throughput of endolysin synthesis should be of sufficient magnitude that the surrounding population of <i>C. difficile</i> diminishes before toxin synthesis occurs significantly.</li>
                  </ul>

                  <h4>
                    After defining the regulatory system requirements above, we then needed to find a suitable regulatory system - either natural or synthetic. As the AIP quorum signalling molecule of <i>C. Difficile </i> is relatively uncharacterised, it would be very difficult to engineer a custom receptor. Thus, it made sense to use the two-component AgrA/AgrC regulatory system already in-built into <i>C. difficile</i>.
                  </h4>

                  <h4>
                    Thus, we chose to mathematically model this AIP detection and two-component AgrA/AgrC regulatory system to provide a model of our corresponding system in vitro.
                  </h4>
                  </div></centre>
              </div>
              </div>
              </div>

              <div class="panel-group">
                <div class="panel panel-default">
                <div class="panel-heading">
                  <h4 class="panel-title">
                    <a data-toggle="collapse" href="#Det">Regulatory System Primer</a>
                  </h4>
                </div>
                <div id="Det" class="panel-collapse collapse">
                  <centre><div class="panel-body">

                      <h4>
                        <i>C. difficile</i> uses a two-component system (TCS) to detect its signalling molecule (AIP). As seen below, when the AIP binds to its AgrC receptor, it triggers a phosphorylation cascade: the receptor first autophosphorylates; this causes the phosphorylation of AgrA protein which then dimerises. This dimerised AgrA protein acts as the specific transcription factor for the expression of CD27L endolysin.
                    </h4> <br>
                        <img src="https://2019.igem.org/wiki/images/e/e6/T--Oxford--Overall_Schematic_Diagram.png" style="width:80%; height:auto; display:block; margin-left:auto; margin-right:auto;">

                  </div>

                    </div></centre>
                </div>
                </div>


                <div class="panel-group">
                    <div class="panel panel-default">
                    <div class="panel-heading">
                      <h4 class="panel-title">
                        <a data-toggle="collapse" href="#ModelAssumptions">Assumptions of Deterministic Model</a>
                      </h4>
                    </div>
                    <div id="ModelAssumptions" class="panel-collapse collapse">
                      <centre><div class="panel-body">
                          <h4>Below are the assumptions:</h4>


                          <ol style="font-size: 18px;">
                          <li>The modelled environment is uniformly mixed, hence spatial dependencies can be ignored.</li>
                          <li>Dephosphorylation of AgrA protein occurs via housekeeping phosphatases.</li>
                          <li>There is negligible decrease in the concentration of dimerised AgrA protein as binding of one molecule of dimerised AgrA protein to the promoter is sufficient to trigger endolysin synthesis.</li>
                          <li>Timescale of degradation of all proteins is approximated to be equal. Consequently, rate of degradation is the same. </li>
                          </ol>
  

                        </div></centre>
                    </div>
                    </div>
                    </div>

                    <div class="panel-group">
                        <div class="panel panel-default">
                        <div class="panel-heading">
                          <h4 class="panel-title">
                            <a data-toggle="collapse" href="#KineticEquations">Kinetic Equations</a>
                          </h4>
                        </div>
                        <div id="KineticEquations" class="panel-collapse collapse">
                          <centre><div class="panel-body">

                              <h3 style="text-align: center;">Production of AgrAC Proteins:</h3>
                                      <br>
                                      <h3>Transcription</h3>
                                   
                            
                              <div class="table-responsive">
                                  <table class="table">
                                    <thead>
                                      <tr>
                                        <th>Kinetic Rate Equation</th>
                                        <th>Kinetic Rate Constant</th>
                                        <th>Kinetic Rate Constant Value Modelled</th>

                                      </tr>
                                    </thead>
                                    <tbody>
                                      <tr>
                                        <td>$$[NULL] \to M$$</td>
                                        <td>$$m$$</td>
                                        <td>$$8.33 \ mRNA \ molecules \ cell^{-1} \ minute^{-1}$$</td>
                                      </tr>
                                      <tr>
                                          <td>$$M \to [NULL]$$</td>
                                          <td>$$\delta_M$$</td>
                                          <td>$$0.05566 \ minutes^{-1}$$</td>
                                      </tr>
                                    </tbody>
                                  </table>
                                </div>

                                <h3>Translation + Membrane Insertion of Cytoplasmic AgrC Receptor</h3>

                        <div class="table-responsive">
                            <table class="table">
                              <thead>
                                <tr>
                                  <th>Kinetic Rate Equation</th>
                                  <th>Kinetic Rate Constant</th>
                                  <th>Kinetic Rate Constant Value Modelled</th>

                                </tr>
                              </thead>
                              <tbody>
                                <tr>
                                  <td>$$M \to M + A$$</td>
                                  <td>$$Ka$$</td>
                                  <td>$$8.33 \times 10^{-8} \ minutes^{-1}$$</td>
                                </tr>

                                <tr>
                                  <td>$$M \to M + C$$</td>
                                  <td>$$Kc$$</td>
                                  <td>$$8.33 \times 10^{-6} \ minutes^{-1}$$</td>
                                </tr>

                                <tr>
                                  <td>$$C \to R$$</td>
                                  <td>$$\alpha_C$$</td>
                                  <td>$$0.0556 \ minutes^{-1}$$</td>
                                </tr>

                                <!-- <tr>
                                  <td>$$A \to [NULL]$$</td>
                                  <td>$$\delta_A$$</td>
                                  <td>$$0.05566 \ minutes^{-1}$$</td>
                                </tr>

                                <tr>
                                    <td>$$C \to [NULL]$$</td>
                                    <td>$$\delta_C$$</td>
                                    <td>$$0.05566 \ minutes^{-1}$$</td>
                                </tr>

                                <tr>
                                  <td>$$R \to [NULL]$$</td>
                                  <td>$$\delta_R$$</td>
                                  <td>$$0.05566 \ minutes^{-1}$$</td> -->
                                </tr>
                              </tbody>
                            </table>
                          </div>


                  <h3 style="text-align: center;">Binding of AIP Signal to Receptor & Subsequent Signal Transduction</h3>
                  <br>
                  <h3>AIP Binding to Transmembrane AgrC Receptor</h3>

                
                  <div class="table-responsive">
                      <table class="table">
                        <thead>
                          <tr>
                            <th>Kinetic Rate Equation</th>
                            <th>Kinetic Rate Constant</th>
                            <th>Kinetic Rate Constant Value Modelled</th>
                            <th>Inverse Kinetic Rate Constant</th>
                            <th>Inverse Kinetic Rate Constant Value Modelled</th>

                          </tr>
                        </thead>
                        <tbody>
                          <tr>
                            <td>$$S + R \leftrightarrow Rb$$</td>
                            <td>$$k_{bind}$$</td>
                            <td>$$1.67 \times 10^{-7} \ (M \times minute)^{-1}$$</td>
                            <td>$$k_{ibind}$$</td>
                            <td>$$1.167 \ minutes^{-1}$$/td>
                          </tr>

                          <!-- <tr>
                            <td>$$Rb \to [NULL]$$</td>
                            <td>$$\delta_{Rb}$$</td>
                            <td></td>
                          </tr> -->


                        </tbody>
                      </table>
                    </div>

                    <h3>Autophosphorylation of AgrC Receptor</h3>
          
            <div class="table-responsive">
                <table class="table">
                  <thead>
                    <tr>
                      <th>Kinetic Rate Equation</th>
                      <th>Kinetic Rate Constant</th>
                      <th>Kinetic Rate Constant Value Modelled</th>
                      <th>Inverse Kinetic Rate Constant</th>
                      <th>Inverse Kinetic Rate Constant Value Modelled</th>

                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>$$Rb \leftrightarrow Rbp$$</td>
                      <td>$$k_{autop}$$</td>
                      <td>$$6.3133 \ minutes^{-1}$$</td>
                      <td>$$k_{iautop}$$</td>
                      <td>$$ $$</td>
                    </tr>


                  </tbody>
                </table>
              </div>

              <h3>Phosphotransfer to AgrA Protein</h3>

    
      <div class="table-responsive">
          <table class="table">
            <thead>
              <tr>
                <th>Kinetic Rate Equation</th>
                <th>Kinetic Rate Constant</th>
                <th>Kinetic Rate Constant Value Modelled</th>
                <th>Inverse Kinetic Rate Constant</th>
                <th>Inverse Kinetic Rate Constant Value Modelled</th>

              </tr>
            </thead>
            <tbody>
              <tr>
                <td>$$Rbp + A \leftrightarrow Rb + Ap$$</td>
                <td>$$k_{phosph}$$</td>
                <td>$$5 \times 10^5 \ minutes^{-1}$$</td>
                <td>$$k_{dphosph}$$</td>
                <td>$$0.1777 \ minutes^{-1}$$</td>
              </tr>


            </tbody>
          </table>
        </div>

        <h3>Dimerisation of AgrA Protein</h3>

<div class="table-responsive">
    <table class="table">
      <thead>
        <tr>
          <th>Kinetic Rate Equation</th>
          <th>Kinetic Rate Constant</th>
          <th>Kinetic Rate Constant Value Modelled</th>
          <th>Inverse Kinetic Rate Constant</th>
          <th>Inverse Kinetic Rate Constant Value Modelled</th>

        </tr>
      </thead>
      <tbody>
        <tr>
          <td>$$2Ap \leftrightarrow Ap_2$$</td>
          <td>$$\phi_d$$</td>
          <td>$$3.33 \times 10^{8} \ (M \times minute)^{-1}$$</td>
          <td>$$\phi_m$$</td>
          <td>$$0.0167 \ minute^{-1}$$</td>
        </tr>


      </tbody>
    </table>
  </div>
  <h3>Degradation of Molecules</h3>

          <div class="table-responsive">
              <table class="table">
                <thead>
                  <tr>
                    <th>Kinetic Rate Equation</th>
                    <th>Kinetic Rate Constant</th>
                    <th>Kinetic Rate Constant Value Modelled</th>

                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>$$X \to NULL \ (where X=B, D, Bt, Dt, S)$$</td>
                    <td>$$\delta_X$$</td>
                    <td>$$0.05566 \ minutes^{-1}$$</td>
                  </tr>


                </tbody>
              </table>
            </div>


                            
                            </div></centre>
                        </div>
                        </div>
                        </div>

                        <div class="panel-group">
                            <div class="panel panel-default">
                            <div class="panel-heading">
                              <h4 class="panel-title">
                                <a data-toggle="collapse" href="#Diff_equations">Differential Equations</a>
                              </h4>
                            </div>
                            <div id="Diff_equations" class="panel-collapse collapse">
                              <centre><div class="panel-body">

                                  <div class="table-responsive">
                                      <table class="table">
                                        <thead>
                                          <tr>
                                            <th>Concentration Modelled</th>
                                            <th>Differential Equation</th>
                                            <th>Process Modelled</th>
                                          </tr>
                                        </thead>
                                        <tbody>
                                          <tr>
                                            <td>mRNA of AgrC Receptor & AgrA Protein</td>
                                            <td>$$\frac {dM}{dt} = m - \delta_{M} \times M$$</td>
                                            <td>Transcription of AgrC & AgrA Genes + Degradation of mRNA</td>
                                          </tr>

                                          <tr>
                                            <td>Deactivated AgrA Protein</td>
                                            <td>$$\frac {dA}{dt} = Ka \times M - \delta_A \times A - k_{phosph} \times Rbp \times A + k_{dphosph} \times Rb \times Ap$$</td>
                                            <td>Translation of AgrA Protein + Degradation of AgrA Protein + Phosphorylation/Dephosphorylation of AgrA </td>
                                          </tr>

                                          <tr>
                                            <td>Cytoplasmic AgrC Receptor</td>
                                            <td>$$\frac {dC}{dt} = Ka \times M - \alpha_C \times C - \delta_C \times C$$</td>
                                            <td>Translation of AgrC Receptor + Membrane Insertion of AgrC Receptor + Degradation of AgrC Receptor</td>
                                          </tr>

                                          <tr>
                                            <td>Unbound AgrC Receptor</td>
                                            <td>$$\frac {dR}{dt} = \alpha_C \times C - \delta_R \times R + k_{ibind} \times Rb$$</td>
                                            <td>Membrane Insertion of AgrC Receptor + Degradation of Inserted AgrC Receptor + Unbinding of AgrC Receptor</td>
                                          </tr>

                                          <tr>
                                            <td>Bound AgrC Receptor</td>
                                            <td>$$\frac {dRb}{dt} = k_{bind} \times S \times R - k_{ibind} \times Rb - \delta_{Rb} \times Rb + k_{iauto} \times Rbp - k_{autop} \times Rb + k_{phosph} \times Rbp \times A$$</td>
                                            <td>Ligand Binding/Unbinding of AgrC Receptor + Degradation of Bound AgrC Receptor + Autophosphorylation/De-autophosphorylation + Phosphotransfer/Dephosphorylation of AgrC Receptor</td>
                                          </tr>

                                          <tr>
                                            <td>Phosphorylated AgrC Receptor</td>
                                            <td>$$\frac {dRbp}{dt} = k_{autop} \times Rb - k_{iauto} \times Rbp - k_{phosph} \times Rbp \times A - \delta_{Rbp} \times Rbp$$</td>
                                            <td> Autophoshorylation/Auto-dephosphorylation of AgrC Receptor + Phosphotransfer/Dephosphorylation to AgrA Protein + Degradation of Phosphorylated AgrC Receptor </td>
                                          </tr>

                                          <tr>
                                            <td>AgrA Protein</td>
                                            <td>$$\frac {dAp}{dt} = k_{phosph} \times Rbp \times A - k_{dphosph} \times Rb \times Ap - 0.5 \times \phi_d \times Ap^2 + \phi_m \times Ap_2 - \delta_{Ap} \times Ap$$</td>
                                            <td>Phosphotransfer/Dephosphorylation of AgrA Protein + Dimerisation/Undimerisation + Degradation of AgrA Protein</td>
                                          </tr>

                                          <tr>
                                            <td>Dimerised AgrA Protein</td>
                                            <td>$$\frac {dAp_2}{dt} = -0.5 \times \phi_d \times Ap \times Ap + \phi_m \times Ap_2 - \delta_{Ap_2} \times Ap_2$$</td>
                                            <td>Dimerisatio/Undimerisation of AgrA Protein + Degradation of Dimerised AgrA Protein</td>
                                          </tr>
                                        </tbody>
                                      </table>
                                    </div>

                                    <div class="panel-group">
                                        <div class="panel panel-default">
                                        <div class="panel-heading">
                                          <h4 class="panel-title">
                                            <a data-toggle="collapse" href="#Upregulation_of_Cell">Upregulation of Cell</a>
                                          </h4>
                                        </div>
                                        <div id="Upregulation_of_Cell" class="panel-collapse collapse">
                                          <centre><div class="panel-body">

                                              <h4>Guided by the mathematical model built by Jabbari et al. (2009)<sup>2</sup>, we constructed a differential equation to track the upregulation state of each cell based on the binding kinetics of the transcription factor. This is further elaborated below.</h4>

                                              <h4>Consider the parameter P that measures the fraction of bacterial population that is upregulated at any given time (P = 1 if the entire population is upregulated, and P =0 if the entire population is downregulated). </h4>

                                              <h4>As per Jabbari et al. (2009)<sup>2</sup>, the following differential equation was used to describe the upregulation of <i>S. aureus</i> cells: </h4>

                                              <h4>$$\frac {dP}{dt} = b \times \frac {Ap}{N} \times (1-P) - u \times P$$</h4>

                                              <h4>The particular parameters are explained in the table below.</h4>

                                              <div class="table-responsive">
                                                  <table class="table">
                                                    <thead>
                                                      <tr>
                                                        <th>Parameter</th>
                                                        <th>Explanation</th>
                                                      </tr>
                                                    </thead>
                                                    <tbody>
                                                      <tr>
                                                      <td>b</td>
                                                      <td>Binding Rate of Transcription Factor </td>
                                                    </tr>

                                                    <tr>
                                                      <td>u</td>
                                                      <td>Unbinding Rate of Transcription Factor</td>
                                                    </tr>

                                                    <tr>
                                                      <td>Ap</td>
                                                      <td>Population-wide Concentration of Transcription Factor in the <i>S. aureus</i> Quorum Sensing System</td>
                                                    </tr>

                                                    <tr>
                                                      <td>N</td>
                                                      <td>Number of Cells</td>
                                                    </tr>
                                               
            
                                                     
                                                    </tbody>
                                                  </table>
                                                </div>

                                                <h4>In the above differential equation, $$b \times Ap \times N \times (1-P)$$ describes the increase in the fraction of upregulated cells due to the binding of the transcription factor to the promoter. Additionally, $$u \times P$$ accounts for the decrease in fraction of upregulated cells due to the unbinding of transcription factor from the promoter.</h4>

                                                <h4>In our particular ProQuorum system, the transcription factor is the dimerised; phosphorylated AgrA protein. Thus, for our particular system, the different equation amounts to:</h4>
                                              <h4>$$\frac {dP}{dt} = b \times \frac {Ap_2}{N} \times (1-P) - u \times P$$</h4>

                                              <h4>The above differential equation describes the upregulation state of the population on a gradient from 0 to 1 based on the binding kinetics of the transcription factor to the promoter.</h4>

                                              <h4>As we are only considering dynamics in a single cell, we set N=1 to obtain: </h4>
                                                <h4>$$\frac {dP}{dt} = b \times Ap_2 \times (1-P) - u \times P$$</h4>

                                                <h4>The above equation describes the upregulation state of a single cell on a gradient of 0 to 1.
                                                    However, given that in reality, a single cell can either be upregulated to produce endolysin or not be upregulated, we must approximate the upregulation state to be an integral value of 0 or 1.
                                                     </h4>

                                                     <h4>To perform such an approximation, we introduce the concept of an upregulation cut-off. The upregulation cut-off refers to the threshold value of P over which a cell will become upregulated and express endolysin. Here, we have taken the upregulation cut-off to be 0.75, or 75 %.</h4>


                                            </div></centre>
                                        </div>
                                        </div>
                                        </div>

                                </div></centre>
                            </div>
                            </div>
                            </div>

                            <div class="panel-group">
                                <div class="panel panel-default">
                                <div class="panel-heading">
                                  <h4 class="panel-title">
                                    <a data-toggle="collapse" href="#Results">Results</a>
                                  </h4>
                                </div>
                                <div id="Results" class="panel-collapse collapse">
                                  <centre><div class="panel-body">
                                      <h4> To analyse the key features of the two-component regulatory system within each cell, we ran cellular-scale simulations in which the cell is exposed to various signal concentrations and patterns.</h4>

                                      <img src="https://2019.igem.org/wiki/images/9/9b/T--oxford--minsignalupregulation.png" style="width:75%; display: block; margin-left: auto; margin-right: auto;" alt="Fig. 1">
                                    <p style="font-size: 14px;">Figure 1. Upregulation Profile for Varying AIP Concentrations</p>

                                    <h4>As depicted in Figure 1 above, AIP concentrations below the concentration of 0.5 nM are unable to reach the 75% upregulation cut-off, and consequently fail to upregulate the cell. </h4>
                                    <h4>Additionally, for AIP concentrations equal to or greater than 0.5 nM, the time taken to reach the upregulation cut-off decreases with increasing AIP concentration, implying that stronger signals take less time to upregulate the cell. </h4>
                                    <br>
                                    <h4>To observe how the concentration of the transcription factor varies during upregulation and downregulation, we simulated an AIP concentration of 1nM at 350 hours and one of 0nM at 650 hours.</h4>
                                    <img src="https://2019.igem.org/wiki/images/1/1a/T--oxford--signaltransductiontcs.png" style="width:75%; display: block; margin-left: auto; margin-right: auto;" alt="Fig. 2">
                                    <p style="font-size: 14px;">Figure 2. Transcription Factor Dynamics, AIP Concentration Modulated</p>
                                    
                                    <h4>As observed in Figure 2 above, the time taken by dimerized phosphorylated AgrA to reach its steady state during upregulation is less than the time taken to reach its steady state during downregulation. </h4>
                                    <h4>The dynamics of AgrA caused by the modulation of AIP concentration as above is also observed to affect the upregulation profile of the cell itself as depicted in Figure 3 below.</h4>
                                    <br>
                                    <img src="https://2019.igem.org/wiki/images/f/fc/T--oxford--singlecellupregulation.png" style="width:75%; display: block; margin-left: auto; margin-right: auto;" alt="Fig. 3">
                                    <p style="font-size: 14px;">Figure 3. Upregulation Profile of Cell, AIP Concentration Modulated </p>
                                    
                                    <h4>Meanwhile, the delay in upregulation of the cell after the AIP signal has been turned on is approximately 33 minutes. Thus, if the duration of active AIP signalling is less than approximately this time, the cell would ultimately not upregulate. </h4>
                                    <br>

                                    <h4>For further insight, we plotted maximum upregulation state of the cell for different signal durations given 1 nM AIP signal concentrations. </h4>

                                    <img src="https://2019.igem.org/wiki/images/4/4f/T--oxford--robustness_single_curves.png" style="width:75%; display: block; margin-left: auto; margin-right: auto;" alt="Fig. 4">
                                    <p style="font-size: 14px;">Figure 4. Upregulation Profile of Cell, Varying AIP Signalling Time </p>
                                    <h4>Figure 4 above illustrates how the cell fails to upregulate for signal durations below 27 minutes, as previously estimated in FIgure 3.</h4>
                                    
                                    <br>
                                    <h4>When running the same simulation for different signal concentrations, we obtain the following graph:</h4>
                                    <img src="https://2019.igem.org/wiki/images/b/b9/T--oxford--robustness_multiplecurves.png" style="width:75%; display: block; margin-left: auto; margin-right: auto;" alt="Fig. 5">
                                    <p style="font-size: 14px;">Figure 5. Upregulation Profile of Cell, Varying AIP Signalling Time with Varying AIP Concentrations</p>
                                    <h4>Figure 5 above indicates that the minimum signal time duration for upregulation decreases with increasing signal concentrations. This is ideal because given that higher concentrations of AIP correspond to larger <i>C. difficile</i> colonies, it is beneficial for our system to react faster than for lower signal concentrations.</h4>
                                    
                                    <br>
                                    <h4>For the mathematical model built so far, the minimum signal concentration required for upregulation is 0.5 nM. However, one of our design requirements was that the probiotic must begin endolysin synthesis before significant toxin synthesis has occurred. </h4>
                                    <h4>To meet this requirement, we could engineer <i>L. reuteri</i> to produce more receptor proteins, thus making it more sensitive to the AIP signal. </h4>
                                    <h4>Thus, we changed the kinetic constant Kc in the mathematical model from $$8.33 \times 10^{-6}$$ per minute per mRNA copy to 8.33 \times 10^{-4}$$ per minute per mRNA copy and ran simulations to test the minimum signal concentration required to upregulate the cell. </h4>
                                    <h4>As seen in Figure 6 below, the cell was ultimately upregulated by AIP signal concentrations in the picomolar range: </h4>
                                    <img src="https://2019.igem.org/wiki/images/3/38/T--oxford--tcs_upregulation_pM.png" style="width:75%; display: block; margin-left: auto; margin-right: auto;" alt="Fig. 6">
                                    <p style="font-size: 14px;">Figure 6. Upregulation Profile of Cell, 3 pM AIP Signal Concentration</p>
                                  </div></centre>
                                </div>
                                </div>
                                </div>

                                
                            <div class="panel-group">
                                <div class="panel panel-default">
                                <div class="panel-heading">
                                  <h4 class="panel-title">
                                    <a data-toggle="collapse" href="#Discussion">Discussion</a>
                                  </h4>
                                </div>
                                <div id="Discussion" class="panel-collapse collapse">
                                  <centre><div class="panel-body">
                                      <h4>
                                          The two-component regulatory system employed by <i>C. difficile</i> has been confirmed via our model to meet the following design requirements:
                                      </h4>
                                      <ul style="font-size: 18px;">
                                        <li>Endolysin expression is triggered only after the concentration of AIP crosses a certain threshold - a valid indicator of <i>C. difficile</i> population density.</li>
                                        <li>Endolysin expression is robust to sudden and short-lived spikes in AIP concentration due to its intrinsic delay in response.</li>
                                      </ul>

                                     <h4>Then, after analyzing the two component regulatory system to be used in our probiotic, we developed our cellular mathematical model into a population wide mathematical model to analyse how our probiotic would cope against a growing <i> C. difficile </i> infection. This is further detailed in our <a href=https://2019.igem.org/Team:Oxford/Model/Therapeutics-Insight>Therapeutic Insight</a>page.</h4>


                                    </div></centre>
                                </div>
                                </div>
                                </div>
          
        </div>
    </div>
  </div>
</section>




 <section id="n5" style="padding-bottom: 15px;">
                                            <div class="container">
                                              <div class="row">
                                                <div class=".col-md-8 .offset-md-2">
                                    
                                                    <h2 style="text-align: center;">
                                                      References
                                                      </h2>
                                    
                                                    <div class="table-responsive">
                                                      <table class="table">
                                                        <thead>
                                                          <tr>
                                                            <th>Reference</th>
                                                          </tr>
                                                        </thead>
                                                        <tbody>
                                                        
                                
                                                          <tr>
                                                            
                                                            <td>Jabbari, Sara, et al. Mathematical Modelling of the Agr Operon in Staphylococcus Aureus. Journal of Mathematical Biology, vol. 61, no. 1, 2009, pp. 1754., doi:10.1007/s00285-009-0291-6.</td>
                                                            
                                                              </td>
                                                          </tr>
                                                        
                                                        </tbody>
                                                      </table>
                                                    </div>
                                                  </div>
                                                </div>
                                              </div>
                                            </section>

    

<style>
    .column {
  float: left;
  width: 25%;
  padding: 5px;
}
</style>

</body>
</html>
{{Oxford/Footer}}
