{{Oxford/Loader}}
{{Oxford}}
<html lang="en">

<!-- THIS IS WHERE THE HTML BEGINS -->

<body>
    <section id="banner">
        <div class="container" style="width:100%; padding-right:0; padding-left:0;">
          <div class="row">
            <div class="col-md-12">
          <div class="inner" style="background-image: url('https://2019.igem.org/wiki/images/6/6b/T--Oxford--Sponsors_Bear.jpeg'); background-size: cover;">
            <div class=image_faded_overlay" style="background-color: lightskyblue; opacity:0.2">
            <div class="translucent-back" style="height: 400px;">
            </div>
          </div>
          <div class="banner_overlay_text">
            Results
          </div>
          </div>
  
        </div>
      </div>
    </div>
        </section>

      <section id="n1">
        <div class="container">
          <div class="row">
            <div class=".col-md-8 .offset-md-2">
  
                <h2> Functionality of &Phi;CD27L Endolysin </h2>

                <h4>The CD27L endolysin is a critical component of our system, as it acts as the agent of <i>C. difficile</i> lysis. Three parts were tested throughout this sub-section of project—they are summarised for clarity in Table 1.</h4>
                <br>
              <h4>Table 1: Different Constructs of CD27L Endolysin in pET28A Expression Vector</h4>
                <div class="table-responsive">
                              <table class="table">
                                <thead>
                                  <tr>
                                    <th>Construct Name</th>
                                    <th>Construct</th>
                                    <th>Part Number</th>
                    
                                  </tr>
                                </thead>
                                <tbody>
                                  <tr>
                                    <td>Dundee-CD27L</td>
                                    <td><i>Hcp</i>-HA_Tag- CD27L<sub>1-179</sub></td>
                                    <td>BBa_K895004</td>
                                  </tr>

                                  <tr>
                                    <td>CD27L</td>
                                    <td>6His-SpyTag-CD27L</td>
                                    <td>BBa_K3183200</td>
                                  </tr>
                                    
                                  <tr>
                                    <td>CD27L<sub>1-179</sub></td>
                                    <td>6His-SpyTag-CD27L<sub>1-179</sub></td>
                                    <td>BBa_K3183201</td>
                                  </tr>
                    
                                </tbody>
                              </table>
                            </div>

          <h4>We wanted to optimise the killing efficacy of the endolysin. Thus, we sought out the endolysin in an existing part made by the 2012 iGEM Team Dundee. This part (Dundee-CD27L<sub>1-179</sub>) was comprised of CD27L<sub>1-179</sub> endolysin linked to the hcp Type VI secretion tag via an haemagglutinin (HA) epitope tag <a href=”http://parts.igem.org/Part:BBa_K895004”>(BBa_K895004)</a>. </h4>
          <h4>After multiple attempts at expression, we were unable to express the protein, with further information on the <a href=”https://2019.igem.org/Improve>Part Improvement</a> page.  Fig. 1 provides a comparison between expression of the full-length endolysin, CD27L and the 2012 Dundee iGEM part BBa_K895004.</h4>
          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#CD27L_Fig1">Fig 1: SDS-PAGE Gel Comparing CD27L Endolysin Samples with Dundee-CD27L Samples</a>
                </h4>
              </div>
              <div id="CD27L_Fig1" class="panel-collapse collapse">
                <centre><div class="panel-body">
                    <img src="" style="width: 100%;" alt="Fig 1: SDS-PAGE Gel Comparing CD27L Endolysin Samples with Dundee-CD27L Samples">
          </div></centre>
          </div>
          </div>
          </div>

          <h4>Regardless, expression of the full-length CD27L expression was successful as shown in Fig.1 and this was quantitatively verified via mass spectrometry<sup>1</sup> as shown in Fig. 2.</h4>
          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#CD27L_Fig2">Fig. 2: Mass Spectrometry of CD27L Endolysin Following Ni-NTA Purification</a>
                </h4>
              </div>
              <div id="CD27L_Fig2" class="panel-collapse collapse">
                <centre><div class="panel-body">
                    <img src="https://2019.igem.org/wiki/images/f/fd/T--Oxford--MassSpectrumResultsPage.png" style="width: 100%;" alt="Fig. 2: Mass Spectrometry of CD27L Endolysin Following Ni-NTA Purification">
          </div></centre>
          </div>
          </div>
          </div>

          <h4>Given this outcome, we decided to express only the truncated CD27L<sub>1-179</sub> endolysin, removing the hcp Type VI secretion tag which we assumed to have interfered with expression. In contrast, successful expression occurred for this new part BBa_K31830012, as shown in Figure 3.</h4>

          <br>
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#CD27L_Fig3">Figure 3: SDS-PAGE  Protein Purification Gels of CD27L & CD27L<sub>1-179</sub> using Ni-NTA Affinity Chromatography</a>
                </h4>
              </div>
              <div id="CD27L_Fig3" class="panel-collapse collapse">
                <centre><div class="panel-body">
                      <div class="container">
                        <div class="row">
                          <div class="col-md-6">
                              <img src="" style="width: 100%;" alt="Fig. 3.1: SDS-PAGE Protein Purification Gel of CD27L using Ni-NTA Affinity Chromatography">
                          </div>
                          <img src="" style="width: 100%;" alt="Fig. 3.2: SDS-PAGE Protein Purification Gel of CD27L<sub>1-179</sub> using Ni-NTA Affinity Chromatography">
                          <div class="col-md-6">


                          </div>
                        </div>
                      </div>

                    </div></centre>
                  </div>
                  </div>
                  </div>

                  <h4>Due to safety concerns, our endolysin killing assays could only be carried out on on <i>Bacillus subitlis</i>, a Category 1 bacteria as opposed to <i>Clostridium difficile</i>, a Category 2 bacteria. <i>B. subtilis</i> has a similar composition of peptidogylcan cell wall to <i>C. difficile</i><sup>2</sup>, allowing us to quantify killing data. </h4>
                   <h4>Figure 4 shows that the CD27L full-length endolysin has greater killing efficacy than the truncated CD27L<sub>1-179</sub>. However, this may be due to a much purer preparation of CD27L compared to CD27L<sub>1-179</sub> as seen in Fig. 3.</h4>
                     <br>

                     <div class="panel-group">
                        <div class="panel panel-default">
                        <div class="panel-heading">
                          <h4 class="panel-title">
                            <a data-toggle="collapse" href="#CD27L_Fig4">Fig. 4: CD27L Versus CD27L<sub>1-179</sub> Endolysin Killing Activity</a>
                          </h4>
                        </div>
                        <div id="CD27L_Fig4" class="panel-collapse collapse">
                          <centre><div class="panel-body">
                            
                            <img src="https://2019.igem.org/wiki/images/f/f8/T--Oxford--TruncatedEndolysinResults.png" style="width:100%" alt="Fig. 4: CD27L Versus CD27L<sub>1-179</sub> Endolysin Killing Activity">
                            <h4>The truncated CD27L<sub>1-179</sub> shows decreased growth relative to the negative control; however, log-phase growth resumes after 150 minutes. 
                            </h4>
                            <h4>CD27L endolysin results in decreased growth during the first 100 minutes, and growth thereafter appears to be linear. This may point to inhibited log-phase growth in subsequent generations of <i>B. subtilis</i> following CD27L exposure.</h4>

          </div></centre>
          </div>
          </div>
          </div>

          <h4>Our next step was then to titrate our killing assay with increasing concentrations of CD27L endolysin. This titration showed measurable effect on <i>B. subtilis</i> cultures in LB broth with final endolysin concentrations greater than 0.2mg/mL (Fig. 5).</h4>

          <h4>From this data, we were able to obtain kinetic constants for endolysin killing, which were subsequently used in our <a>href=”https://2019.igem.org/Team:Oxford/Model/Design>mathematical models</a>.</h4>

          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#CD27L_Fig5">Fig. 5: CD27L Endolysin Killing Assay Varying Final Endolysin Concentration</a>
                </h4>
              </div>
              <div id="CD27L_Fig5" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  
                  <img src="https://2019.igem.org/wiki/images/6/6b/T--Oxford--EndolysinTitration.png" style="width:100%" alt="FFig. 5: CD27L Endolysin Killing Assay Varying Final Endolysin Concentration">
               <h4>Here we show that the effect of the endolysin appears to inhibit growth at concentrations above 0.002mg/mL. Growth seems to increase significantly at 150 minutes, which may be indicative of CD27L degradation. </h4>
               <h4>Interestingly, 0.2mg/mL of endolysin grows linearly for duration of the assay (over 6 hours), as does 0.45mg/mL after 150 minutes. This may mean that log-phase growth is inhibited under these conditions.</h4>

</div></centre>
</div>
</div>
</div>

  <h4>Moreover, we demonstrated that CD27L can be denatured by heat, indicating that protein folding is critical for its lytic effect.</h4>

  <div class="panel-group">
      <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a data-toggle="collapse" href="#CD27L_Fig6">Fig. 6: Effect of 10 Minute Temperature Incubation on CD27L Endolysin Efficacy</a>
        </h4>
      </div>
      <div id="CD27L_Fig6" class="panel-collapse collapse">
        <centre><div class="panel-body">
          
          <img src="" style="width:100%" alt="Fig. 6: Effect of 10 Minute Temperature Incubation on CD27L Endolysin Efficacy">
     

</div></centre>
</div>
</div>
</div>

  <h4>We also showed that endolysin was specific to <i>B. subtilis</i> using a killing assay involving <i>E. coli</i>, <i>B. subtilis</i>, and our chassis <i>Lactobacillus reuteri</i> (Fig. 7).</h4>

  <div class="panel-group">
      <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a data-toggle="collapse" href="#CD27L_Fig7">Fig. 7: CD27L Endolysin Killing Assay on <i>E. coli</i>, <i>Lactobacillus reuteri</i>, and <i>Bacillus subtilis</i></a>
        </h4>
      </div>
      <div id="CD27L_Fig7" class="panel-collapse collapse">
        <centre><div class="panel-body">
          
          <img src="" style="width:100%" alt="Fig. 7: CD27L Endolysin Killing Assay on <i>E. coli</i>, <i>Lactobacillus reuteri</i>, and <i>Bacillus subtilis</i>">
     

</div></centre>
</div>
</div>
</div>  

    <h4>Finally, scanning electron microscopy (SEM) was performed with the aid of Errin Johnson at the Dunn School of Pathology. Below shows images of <i>B. subtilis</i> following CD27L endolysin treatment (Fig. 8).</h4>

    <h4>We also showed that endolysin was specific to <i>B. subtilis</i> using a killing assay involving <i>E. coli</i>, <i>B. subtilis</i>, and our chassis <i>Lactobacillus reuteri</i> (Fig. 7).</h4>

    <div class="panel-group">
        <div class="panel panel-default">
        <div class="panel-heading">
          <h4 class="panel-title">
            <a data-toggle="collapse" href="#CD27L_Fig7">Fig. 7: CD27L Endolysin Killing Assay on <i>E. coli</i>, <i>Lactobacillus reuteri</i>, and <i>Bacillus subtilis</i></a>
          </h4>
        </div>
        <div id="CD27L_Fig7" class="panel-collapse collapse">
          <centre><div class="panel-body">
            
            <img src="" style="width:100%" alt="Fig. 7: CD27L Endolysin Killing Assay on <i>E. coli</i>, <i>Lactobacillus reuteri</i>, and <i>Bacillus subtilis</i>">
       
            <h4>Here, the SEM images show slightly deflated <i>B. subtilis</i> cells and cells with small bulges on them compared to the control. This may be due to compromised structure of the peptidoglycan scaffold and consequent bulging of the cell membrane. However, this will have to be explored further to definitively confirm the effects of CD27L endolysin on <i>B. subtilis</i> morphology.</h4>
  </div></centre>
  </div>
  </div>
  </div>  

</div>
</div>
</div>
</section>

<section id="n2" style="background-color: #f2f2f2ff; padding-bottom:15px;">
  <div class="container">
    <div class="row">
      <div class=".col-md-8 .offset-md-2">
          <h2><i>Lactobacillus reuteri</i></h2>

          <h3>Transformation via Electroporation</h3>

          <h4>Electroporation is the standard method of transformation for <i>Lactobacillus</i> spp., with several well-established protocols existent in literature, such as the ones by Berthier<sup>1</sup>, Aukrust<sup>2</sup>, Spengler<sup>3</sup> etc. However, in spite of the previous success with such procedures, our attempts to transform the <i>L. reuteri</i> type strain, DSM20016, were never successful. </h4>

          <h4>As it was later revealed, in order for such a transformation to have worked, the plasmid DNA should have been isolated from <i>L. lactis</i>, rather than <i>E. coli</i>. This is because the <i>L. lactis</i>-derived plasmid methylation pattern prevents its digestion by the host bacterium.</h4>

          <h4>While amplification in <i>L. lactis</i> would not have been feasible at that stage in the project, we acquired a strain of <i>L. reuteri</i> 100-23c and a new transformation protocol (which can be found in our <a href=”https://2019.igem.org/Team:Oxford/Protocols>Protocols</a> page) from the Quadram Institute. From that point forward, our transformations succeeded. As proof, we performed colony PCR on transformants containing our control plasmid, pTRKH3-GFP, as well as the other constructs of CD27L (Figures 1, 3, 8).</h4>
        <br>
      
          <div class="panel-group">
              <div class="panel panel-default">
              <div class="panel-heading">
                <h4 class="panel-title">
                  <a data-toggle="collapse" href="#L_reuteri_Fig1">Fig. 1: pTRKH3-GFP <i>L. reuteri</i> Transformants Colony PCR</i></a>
                </h4>
              </div>
              <div id="L_reuteri_Fig1" class="panel-collapse collapse">
                <centre><div class="panel-body">
                  
                  <img src="https://2019.igem.org/wiki/images/b/b2/T--Oxford--pTRKH3_GFP_colony_PCR.png" style="width:100%" alt="Fig. 1: pTRKH3-GFP <i>L. reuteri</i> Transformants Colony PCR">
             
                  <p style="font-size: 14px;">Conditions: Primers - S01 and S02 (our generic pTRKH3 sequencing primers). Cell lysate as sample template, and miniprepped vector as control template. Annealing temperature: 55 ⁰C, extension time: 2:00. NEB Taq polymerase 2x master mix.</p>
        </div></centre>
        </div>
        </div>
        </div>  
      
      
        </div>
    </div>
  </div>
</section>

<section id="n6">>
    <div class="container">
      <div class="row">
        <div class=".col-md-8 .offset-md-2">

            <h2 style="text-align: center;">
              References
              </h2>

            <div class="table-responsive">
              <table class="table">
                <thead>
                  <tr>
                    <th> #</th>
                    <th>Reference</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>1</td>
                    <td>Dunne, Matthew et al. “The CD27L and CTP1L endolysins targeting Clostridia contain a built-in trigger and release factor.” PLoS pathogens vol. 10,7 e1004228. 24 Jul. 2014, doi:10.1371/journal.ppat.1004228</td>
                      
                  </tr>

                  <tr>
                    <td>2</td>
                    <td>Schleifer, K H, and O Kandler. “Peptidoglycan types of bacterial cell walls and their taxonomic implications.” Bacteriological reviews vol. 36,4 (1972): 407-77.</td>
                  </tr>
                  <tr>
                    <td>3</td>
                    <td>Berthier F. Zagorec M. Champomier-Vergès M. Ehrlich S.D. Morel-Deville F. (1996) Efficient transformation of Lactobacillus sake by electroporation. Microbiology 142, 1273–1279.</td>
                  </tr>

                  <tr>
                    <td>4</td>
                    <td>Aukrust, T. W., M. B. Brurberg, and I. F. Nes. 1995. Transformation of Lactobacillus by electroporation. Methods Mol. Biol. 47:201-208</td>
                      
                  </tr>
                  
                  <tr>
                    <td>5</td>
                    <td>Joseph R Spangler, Julie C Caruana, Daniel A Phillips, Scott A Walper, Broad range shuttle vector construction and promoter evaluation for the use of Lactobacillus plantarum WCFS1 as a microbial engineering platform, Synthetic Biology, Volume 4, Issue 1, 2019, ysz012,</td>
                  </tr>

                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </section>

<style>
    .column {
  float: left;
  width: 25%;
  padding: 5px;
}
</style>

</body>
</html>
{{Oxford/Footer}}
